<!--START OF HEADER - DON'T ALTER --><HTML><HEAD>  <TITLE> EMBOSS: gaaui </TITLE></HEAD><BODY BGCOLOR="#FFFFFF" text="#000000"><table align=center border=0 cellspacing=0 cellpadding=0><tr><td valign=top><A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="http://soap.g-language.org/gembassy/emboss_explorer/manual/emboss_icon.jpg" alt="" width=150 height=48></a></td><td align=left valign=middle><b><font size="+6">gaaui</font></b></td></tr></table><br>&nbsp;<p><!--END OF HEADER--><H2> Function </H2>   Calculates various indece of amino acid usage<!--DON'T WRITE ANYTHING HERE.IT IS DONE FOR YOU.--><H2>Description</H2><p>   gaaui calculates amino acid usage indices for proteins (excluding<br />   formylmethionine). Calculated indices are as follows,<br />      Laa:   Length in amino acids<br />      ndaa:  Number of different amino acids<br />      Haau:  Entropy of amino acid usage<br />      mmw:   Mean molecular weight<br />      gravy: Mean hydropathic indices of each amino acid<br />      aroma: Relative frequency of aromatic amino acids<br /><br />   G-language SOAP service is provided by the<br />   Institute for Advanced Biosciences, Keio University.<br />   The original web service is located at the following URL:<br /><br />   http://www.g-language.org/wiki/soap<br /><br />   WSDL(RPC/Encoded) file is located at:<br /><br />   http://soap.g-language.org/g-language.wsdl<br /><br />   Documentation on G-language Genome Analysis Environment methods are<br />   provided at the Document Center<br /><br />   http://ws.g-language.org/gdoc/<br /><br /></p><H2>Usage</H2>Here is a sample session with gaaui<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>% gaaui refseqn:NC_000913Calculates various indece of amino acid usageAAINDEX entry output file [nc_000913.gaaui]: </pre></td></tr></table>Go to the <a href="#input">input files</a> for this example<br>Go to the <a href="#output">output files</a> for this example<br><br><h2>Command line arguments</h2><table border cellspacing=0 cellpadding=3 bgcolor="#ccccff"><tr bgcolor="#FFFFCC"><th align="left">Qualifier</th><th align="left">Type</th><th align="left">Description</th><th align="left">Allowed values</th><th align="left">Default</th></tr><tr bgcolor="#FFFFCC"><th align="left" colspan=5>Standard (Mandatory) qualifiers</th></tr><tr bgcolor="#FFFFCC"><td>[-sequence]<br>(Parameter 1)</td><td>seqall</td><td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td><td>Readable sequence(s)</td><td><b>Required</b></td></tr><tr bgcolor="#FFFFCC"><td>[-outfile]<br>(Parameter 2)</td><td>outfile</td><td>AAINDEX entry output file</td><td>Output file</td><td><i>&lt;*&gt;</i>.gaaui</td></tr><tr bgcolor="#FFFFCC"><th align="left" colspan=5>Additional (Optional) qualifiers</th></tr><tr><td colspan=5>(none)</td></tr><tr bgcolor="#FFFFCC"><th align="left" colspan=5>Advanced (Unprompted) qualifiers</th></tr><tr bgcolor="#FFFFCC"><td>-[no]accid</td><td>boolean</td><td>Include to use sequence accession ID as query</td><td>Boolean value Yes/No</td><td>Yes</td></tr></table><h2 id="input">Input file format</h2><p>   The database definitions for following commands are available at<br />   http://soap.g-language.org/kbws/embossrc<br /><br />   gaaui reads one or more nucleotide sequences.<br /><br /></p><h2 id="output">Output file format</h2><p>   The output from gaaui is to a plain text file.<br /><br />   File: nc_000913.gaaui<br /><br /><table width="90%"><tr><td bgcolor="#CCFFCC">Sequence: NC_000913<br />Laa,ndaa,Haau,mmw,gravy,aroma,gene<br />20,8,2.4842,117.48,+0.0150,0.0000,thrL<br />819,20,4.0887,126.65,+0.0328,0.0659,thrA<br />309,20,4.1228,126.35,+0.0181,0.0712,thrB<br />427,20,4.0806,128.00,-0.1014,0.0843,thrC<br />97,18,3.9165,133.54,-1.0268,0.0928,yaaX<br />257,19,4.0733,132.55,-0.4117,0.1089,yaaA<br />475,20,4.0413,126.46,+0.6781,0.1242,yaaJ<br />316,20,4.0395,128.99,-0.2165,0.0728,talB<br /><br /><font color=red>[Part of this file has been deleted for brevity]</font><br /><br />169,20,4.0001,124.90,+0.0231,0.0710,yjjX<br />214,20,3.9937,129.77,-0.3813,0.0374,ytjC<br />288,20,4.1421,132.58,-0.3628,0.1111,rob<br />156,20,4.0627,126.72,-0.0442,0.0705,creA<br />228,20,4.0471,131.94,-0.1408,0.0789,creB<br />473,20,4.0254,128.01,+0.0023,0.0677,creC<br />449,20,4.0871,128.66,+0.2082,0.0980,creD<br />237,20,4.0729,132.54,-0.4970,0.0675,arcA<br />45,15,3.5800,123.27,+0.7533,0.0222,yjjY<br />227,20,4.0283,128.63,-0.0031,0.0573,yjtD<br /></td></tr></table><br /><br /></p><h2>Data files</h2><p>None.</p><h2>Notes</h2><p>None.</p><h2>References</h2><pre>   Lobry, JR. and Gautier, C. (1994) Hydrophobicity, expressivity and      aromaticity are the major trends of amino-acid usage in 999 Escherichia      coli chromosome-encoded genes, Nucleic Acids Res, 22:3174-3180.a   Zavala A et al. (2002) Trends in codon and amino acid usage in Thermotoga      maritima J Mol Evol. 54(5):563-8.   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.   Arakawa, K. and Tomita, M. (2006) G-language System as a Platform for      large-scale analysis of high-throughput omics data, J. Pest Sci.,      31, 7.   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome      Analysis Environment with REST and SOAP Web Service Interfaces,      Nucleic Acids Res., 38, W700-W705.</pre><h2>Warnings</h2><p>None.</p><h2>Diagnostic Error Messages</h2><p>None.</p><h2>Exit status</h2><p>It always exits with a status of 0.</p><h2>Known bugs</h2><p>None.</p><h2>See also</h2><table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th><th>Description</th></tr><tr><td><a href="gaminoinfo.html">gaminoinfo</a></td><td>Prints out basic amino acid sequence statistics</td></tr><tr><td><a href="gcodoncompiler.html">gcodoncompiler</a></td><td>Calculate various kinds of amino acid and codon usage data</td></tr></table><h2>Author(s)</h2><pre>Hidetoshi Itaya (celery@g-language.org)  Institute for Advanced Biosciences, Keio University  252-0882 JapanKazuharu Arakawa (gaou@sfc.keio.ac.jp)  Institute for Advanced Biosciences, Keio University  252-0882 Japan</pre><h2>History</h2>   2012 - Written by Hidetoshi Itaya<h2>Target users</h2>   This program is intended to be used by everyone and everything, from   naive users to embedded scrips.<h2>Comments</h2>   None.