<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gamino_info </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="/gembassy/emboss_explorer/manual/emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gamino_info
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Prints out basic amino acid sequence statistics
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   This method prints out basic compositional statistics of the <br>
   given amino acid sequence in a human readable manner.<br>
<br>
   Displayed information includes the following:<br>
   Molecular weight, number of residues, average residue weight, charge,<br>
   isoelectric point, number/mole/DayhoffStat of each amino acid, and<br>
   percentage of Tiny (A+C+G+S+T), Small (A+B+C+D+G+N+P+S+T+V), <br>
   Aliphatic (I+L+V), Aromatic (F+H+W+Y), Non-polar (A+C+F+G+I+L+M+P+V+W+Y), <br>
   Polar (D+E+H+K+N+Q+R+S+T+Z), Charged (B+D+E+H+K+R+Z), Basic (H+K+R),<br>
   Acidic (B+D+E+Z) residues.<br>
  <br>
   G-language SOAP service is provided by the<br>
   Institute for Advanced Biosciences, Keio University.<br>
   The original web service is located at the following URL:<br>
<br>
   http://www.g-language.org/wiki/soap<br>
<br>
   WSDL(RPC/Encoded) file is located at:<br>
<br>
   http://soap.g-language.org/g-language.wsdl<br>
<br>
   Documentation on G-language Genome Analysis Environment methods are<br>
   provided at the Document Center<br>
<br>
   http://ws.g-language.org/gdoc/
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with gamino_info

% gamino_info -show ADH_HUMAN.fasta
Prints out basic amino acid sequence statistics
Sequence: 
AminoInfo of  from 1 to 406

Molecular weight = 47725.11 Residues = 406
Average Weight = 117.55 Charge = 8
Isoelectric Point = 8.0134
Residue         Number          Mole%               DayhoffStat

 =             1               0.246                     0.000
. =             2               0.493                     0.000
0 =             6               1.478                     0.000
2 =             1               0.246                     0.000
4 =             1               0.246                     0.000
5 =             1               0.246                     0.000
6 =             3               0.739                     0.000
8 =             1               0.246                     0.000
9 =             1               0.246                     0.000
> =             1               0.246                     0.000
A = Ala         31              7.635                     0.888
C = Cys         16              3.941                     1.359
D = Asp         19              4.680                     0.851
E = Glu         19              4.680                     0.780
F = Phe         16              3.941                     1.095
G = Gly         38              9.360                     1.114
H = His         8               1.970                     0.985
I = Ile         23              5.665                     1.259
K = Lys         32              7.882                     1.194
L = Leu         29              7.143                     0.965
M = Met         9               2.217                     1.304
N = Asn         14              3.448                     0.802
P = Pro         21              5.172                     0.995
Q = Gln         6               1.478                     0.379
R = Arg         10              2.463                     0.503
S = Ser         22              5.419                     0.774
T = Thr         24              5.911                     0.969
V = Val         39              9.606                     1.455
W = Trp         2               0.493                     0.379
Y = Tyr         6               1.478                     0.435
_ =             4               0.985                     0.000

Property        Residues                Number          Mole%
Tiny            (A+C+G+S+T)             131             32.266
Small           (A+B+C+D+G+N+P+S+T+V)   224             55.172
Aliphatic       (I+L+V)                 91              22.414
Aromatic        (F+H+W+Y)               32              7.882
Non-polar       (A+C+F+G+I+L+M+P+V+W+Y) 230             56.650
Polar           (D+E+H+K+N+Q+R+S+T+Z)   154             37.931
Charged         (B+D+E+H+K+R+Z)         88              21.675
Basic           (H+K+R)                 50              12.315
Acidic          (B+D+E+Z)               38              9.360

</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Protein sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-outfile]<br>(Parameter 2)</td>
<td>outfile</td>
<td>Output file name</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.gamino_info</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   gamino_info reads nucleotide sequences.
</p>

<h2>Output file format</h2>

<p>
   The output from gamino_info is to a file or stdout.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platform for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Web Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>

<h2>Warnings</h2>

<p>
None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

<tr>
<td><a href="gaaui.html">gaaui</a></td>
<td>Calculates various indece of amino acid usage</td>
</tr>

<tr>
<td><a href="gcodon_compiler.html">gcodon_compiler</a></td>
<td>Calculate various kinds of amino acid and codon usage data</td>
</tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa(gaou@g-language.org) 
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

