<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gamino_info </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gamino_info
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
Prints out basic amino acid sequence statistics
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   This method prints out basic compositional statistics of the 
   given amino acid sequence in a human readable manner.

   Displayed information includes the following:
   Molecular weight, number of residues, average residue weight, charge,
   isoelectric point, number/mole/DayhoffStat of each amino acid, and
   percentage of Tiny (A+C+G+S+T), Small (A+B+C+D+G+N+P+S+T+V), 
   Aliphatic (I+L+V), Aromatic (F+H+W+Y), Non-polar (A+C+F+G+I+L+M+P+V+W+Y), 
   Polar (D+E+H+K+N+Q+R+S+T+Z), Charged (B+D+E+H+K+R+Z), Basic (H+K+R),
   Acidic (B+D+E+Z) residues.

  
</p>

<p>
G-language SOAP service is provided by the Institute for Advanced Biosciences, Keio University.<br />

The original web service is located at the following URL:<br />
http://www.g-language.org/wiki/soap
</p>

<p>
WSDL(RPC/Encoded) file is located at:<br />
http://soap.g-language.org/g-language.wsdl
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with gamino_info

% gamino_info NC_000913.gbk
NC_000913
AminoInfo of  from 1 to 4639686

Molecular weight = 468940064.45 Residues = 4639686
Average Weight = 101.07 Charge = 0
Isoelectric Point = 2.4016
Residue         Number          Mole%               DayhoffStat

 =             1               0.000                     0.000
0 =             3               0.000                     0.000
1 =             1               0.000                     0.000
3 =             1               0.000                     0.000
9 =             1               0.000                     0.000
> =             1               0.000                     0.000
A = Ala         1142228         24.619                    2.863
C = Cys         1179555         25.423                    8.767
G = Gly         1176923         25.366                    3.020
N = Asn         1               0.000                     0.000
T = Thr         1140970         24.592                    4.031
_ =             1               0.000                     0.000

Property        Residues                Number          Mole%
Tiny            (A+C+G+S+T)             4639676         100.000
Small           (A+B+C+D+G+N+P+S+T+V)   4639677         100.000
Aliphatic       (I+L+V)                 0               0.000
Aromatic        (F+H+W+Y)               0               0.000
Non-polar       (A+C+F+G+I+L+M+P+V+W+Y) 3498706         75.408
Polar           (D+E+H+K+N+Q+R+S+T+Z)   1140971         24.592
Charged         (B+D+E+H+K+R+Z)         0               0.000
Basic           (H+K+R)                 0               0.000
Acidic          (B+D+E+Z)               0               0.000



</pre></td></tr></table>



<h2>Command line argumants</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Protein sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-accid</td>
<td>boolean</td>
<td>Include to use sequence accession ID as query</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
gamino_info reads nucleotide sequences.
</p>

<h2>Ouptput file format</h2>

<p>
The output from gamino_info is to standard output.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>

   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Wev Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>
<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa (gaou@sfc.keio.ac.jp)
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

    None.

