<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gbaseinfocontent </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emb
oss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gbaseinfocontent
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
Calculates and graphs the sequence conservation using information content
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   This function calculates and graphs the sequence conservation in regions 
   around the start/stop codons using information content.
</p>

<p>
G-language SOAP service is provided by the Institute for Advanced Biosciences, Keio University.<br />
The original web service is located at the following URL:<br />
http://www.g-language.org/wiki/soap
</p>

<p>
WSDL(RPC/Encoded) file is located at:<br />
http://soap.g-language.org/g-language.wsdl
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with gbaseinfocontent

% gbaseinfocontent 
Calculates and graphs the sequence conservation using information content
Input nucleotide sequence: input.fna
-30     2.46682
-29     2.46974
-28     2.47447
-27     2.47456
-26     2.48113
-25     2.47919
-24     2.48432
-23     2.49833
-22     2.50999
-21     2.51332
-20     2.51344
-19     2.51057
-18     2.51365
-17     2.52460
-16     2.52559
-15     2.52502
-14     2.54766
-13     2.64310
-12     2.80771
-11     3.01463
-10     2.78745
-9      2.90541
-8      2.68756
-7      2.56350
-6      2.53270
-5      2.56253
-4      2.56363
-3      2.56254
-2      1.98481
-1      1.04509
0       1.66340
1       0.91749
2       0.90700
3       2.60109
4       2.61916
5       2.63561
6       2.44869
7       2.53325
8       2.54410
9       2.36641
10      2.53615
11      2.55475
12      2.35462
13      2.52858
14      2.52296
15      2.49355
16      2.51738
17      2.50030
18      2.32022
19      2.50154
20      2.50607
21      2.30714
22      2.50378
23      2.50487
24      2.32515
25      2.50308
26      2.49580
27      2.31593
28      2.49287
29      2.50863
30      2.42663

</pre></td></tr></table>



<h2>Command line argumants</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>sequence</td>
<td>Nucleotide sequence filename and optional format, or reference (input USA)</td>
<td>Readable sequence</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-position</td>
<td>string</td>
<td>either 'start' (around start codon) or 'end' (around stop codon) to create the PWM</td>
<td>Any string</td>
<td>start</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-upstream</td>
<td>integer</td>
<td>Length upstream of specified position to create PWM</td>
<td>Any integer value</td>
<td>30</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-downstream</td>
<td>integer</td>
<td>Length downstream of specified position to create PWM</td>
<td>Any integer value</td>
<td>30</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-patlen</td>
<td>integer</td>
<td>Length of oligomer to count</td>
<td>Any integer value</td>
<td>3</td>
</tr>

</table>

<h2>Input file format</h2>

<p>
gbaseinfocontent reads inputtype.
</p>

<h2>Ouptput file format</h2>

<p>
The output from gbaseinfocontent is to outputtype.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.                     

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Wev Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>

<h2>Warnings</h2>

<p>
None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery @ g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Koya Mori(mory@g-language.org)
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

