<!--START OF HEADER - DON'T ALTER --><HTML><HEAD>  <TITLE> EMBOSS: gbaseinformationcontent </TITLE></HEAD><BODY BGCOLOR="#FFFFFF" text="#000000"><table align=center border=0 cellspacing=0 cellpadding=0><tr><td valign=top><A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="http://soap.g-language.org/gembassy/emboss_explorer/manual/emboss_icon.jpg" alt="" width=150 height=48></a></td><td align=left valign=middle><b><font size="+6">gbaseinformationcontent</font></b></td></tr></table><br>&nbsp;<p><!--END OF HEADER--><H2> Function </H2>   Calculates and graphs the sequence conservation using information content<!--DON'T WRITE ANYTHING HERE.IT IS DONE FOR YOU.--><H2>Description</H2><p>   This function calculates and graphs the sequence conservation in regions<br />   around the start/stop codons using information content. Values are obtained<br />   by subtracting the entropy for each positfion from the maximum possible value<br />   (which will be 2 in the case of nucleotide sequences). Information content<br />   will show the highest value when the frequency is most biased to a single<br />   alphabet.<br />    <br />   G-language SOAP service is provided by the<br />   Institute for Advanced Biosciences, Keio University.<br />   The original web service is located at the following URL:<br /><br />   http://www.g-language.org/wiki/soap<br /><br />   WSDL(RPC/Encoded) file is located at:<br /><br />   http://soap.g-language.org/g-language.wsdl<br /><br />   Documentation on G-language Genome Analysis Environment methods are<br />   provided at the Document Center<br /><br />   http://ws.g-language.org/gdoc/<br /><br /></p><H2>Usage</H2>Here is a sample session with gbaseinformationcontent<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>% gbaseinformationcontent refseqn:NC_000913Calculates and graphs the sequence conservation using information contentProgram compseq output file (optional) [nc_000913.gbaseinformationcontent]: </pre></td></tr></table>Go to the <a href="#input">input files</a> for this example<br>Go to the <a href="#output">output files</a> for this example<br><br>   Example 2<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>% gbaseinformationcontent refseqn:NC_000913 -plot -graph pngCalculates and graphs the sequence conservation using information contentCreated gbaseinformationcontent.1.png</pre></td></tr></table>Go to the <a href="#input">input files</a> for this example<br>Go to the <a href="#output">output files</a> for this example<br><br><h2>Command line arguments</h2><table border cellspacing=0 cellpadding=3 bgcolor="#ccccff"><tr bgcolor="#FFFFCC"><th align="left">Qualifier</th><th align="left">Type</th><th align="left">Description</th><th align="left">Allowed values</th><th align="left">Default</th></tr><tr bgcolor="#FFFFCC"><th align="left" colspan=5>Standard (Mandatory) qualifiers</th></tr><tr bgcolor="#FFFFCC"><td>[-sequence]<br>(Parameter 1)</td><td>seqall</td><td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td><td>Readable sequence(s)</td><td><b>Required</b></td></tr><tr bgcolor="#FFFFCC"><td>-graph</td><td>xygraph</td><td>Graph type</td><td>EMBOSS has a list of known devices, including ps, hpgl, hp7470, hp7580, meta, cps, x11, tek, tekt, none, data, xterm, png, gif, svg</td><td><i>EMBOSS_GRAPHICS</i> value, or x11</td></tr><tr bgcolor="#FFFFCC"><td>-outfile</td><td>outfile</td><td>Program compseq output file (optional)</td><td>Output file</td><td><i>&lt;*&gt;</i>.gbaseinformationcontent</td></tr><tr bgcolor="#FFFFCC"><th align="left" colspan=5>Additional (Optional) qualifiers</th></tr><tr><td colspan=5>(none)</td></tr><tr bgcolor="#FFFFCC"><th align="left" colspan=5>Advanced (Unprompted) qualifiers</th></tr><tr bgcolor="#FFFFCC"><td>-position</td><td>selection</td><td>Either 'start' (around start codon) or 'end' (around stop codon) to create the PWM</td><td>Choose from selection list of values</td><td>start</td></tr><tr bgcolor="#FFFFCC"><td>-upstream</td><td>integer</td><td>Length upstream of specified position to create PWM</td><td>Any integer value</td><td>30</td></tr><tr bgcolor="#FFFFCC"><td>-downstream</td><td>integer</td><td>Length downstream of specified position to create PWM</td><td>Any integer value</td><td>30</td></tr><tr bgcolor="#FFFFCC"><td>-patlen</td><td>integer</td><td>Length of oligomer to count</td><td>Any integer value</td><td>3</td></tr><tr bgcolor="#FFFFCC"><td>-[no]accid</td><td>boolean</td><td>Include to use sequence accession ID as query</td><td>Boolean value Yes/No</td><td>Yes</td></tr><tr bgcolor="#FFFFCC"><td>-plot</td><td>toggle</td><td>Include to plot result</td><td>Toggle value Yes/No</td><td>No</td></tr></table><h2 id="input">Input file format</h2><p>   The database definitions for following commands are available at<br />   http://soap.g-language.org/kbws/embossrc<br /><br />   gbaseinformationcontent reads one or more nucleotide sequences.<br /><br /></p><h2 id="output">Output file format</h2><p>   The output from gbaseinformationcontent is to a plain text file or the EMBOSS graphics device.<br /><br />   File: nc_000913.gbaseinformationcontent<br /><br /><table width="90%"><tr><td bgcolor="#CCFFCC">Sequence: NC_000913<br />-30,2.42457<br />-29,2.42811<br />-28,2.43235<br />-27,2.43116<br />-26,2.44278<br />-25,2.44236<br />-24,2.44502<br />-23,2.46097<br />-22,2.46588<br /><br /><font color=red>[Part of this file has been deleted for brevity]</font><br /><br />21,2.27547<br />22,2.46974<br />23,2.46342<br />24,2.32686<br />25,2.46245<br />26,2.46061<br />27,2.27664<br />28,2.45650<br />29,2.48206<br />30,2.29140<br /></td></tr></table></p><h2>Data files</h2><p>None.</p><h2>Notes</h2><p>None.</p><h2>References</h2><pre>   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.   Arakawa, K. and Tomita, M. (2006) G-language System as a Platform for      large-scale analysis of high-throughput omics data, J. Pest Sci.,      31, 7.   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome      Analysis Environment with REST and SOAP Web Service Interfaces,      Nucleic Acids Res., 38, W700-W705.</pre><h2>Warnings</h2><p>None.</p><h2>Diagnostic Error Messages</h2><p>None.</p><h2>Exit status</h2><p>It always exits with a status of 0.</p><h2>Known bugs</h2><p>None.</p><h2>See also</h2><table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th><th>Description</th></tr><tr><td><a href="gbaseentropy.html">gbaseentropy</a></td><td>Calculates and graphs the sequence conservation using Shanon uncertainty (entropy)</td></tr><tr><td><a href="gbaserelativeentropy.html">gbaserelativeentropy</a></td><td>Calculates and graphs the sequence conservation using Kullback-Leibler divergence (relative entropy)</td></tr></table><h2>Author(s)</h2><pre>Hidetoshi Itaya (celery@g-language.org)  Institute for Advanced Biosciences, Keio University  252-0882 JapanKazuharu Arakawa (gaou@sfc.keio.ac.jp)  Institute for Advanced Biosciences, Keio University  252-0882 Japan</pre><h2>History</h2>   2012 - Written by Hidetoshi Itaya<h2>Target users</h2>   This program is intended to be used by everyone and everything, from   naive users to embedded scrips.<h2>Comments</h2>   None.</BODY></HTML>