<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gbui </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gbui
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
Calculates base usage indices for protein-coding sequences
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   This program calculates base usage indices for protein-coding sequences
   (excluding start and stop codons), and inputs in the G instance.
   Informations stored in the G instance are:
   $gb->{$id}->{"acgt$p"}: total number of bases (A+T+G+C)
   $gb->{$id}->{"ryr$p"}: purine/pyrimidine ratio (A+G)/(T+C)
   $gb->{$id}->{"gac$p"}: G+A content (G+A)/(A+T+G+C)
   $gb->{$id}->{"gcc$p"}: G+C content (G+C)/(A+T+G+C)
   $gb->{$id}->{"gtc$p"}: G+T content (G+T)/(A+T+G+C)
   $gb->{$id}->{"gcs$p"}: GC skew (G-C)/(G+C)
   $gb->{$id}->{"tas$p"}: TA skew (T-A)/(T+A)
   $gb->{$id}->{"Hbu$p"}: entropy of base usage
   $gb->{header_bui}

  Usage:
   NULL = &bui(G instance);

  Options:
   -output   output option (default: 'stdout')
   'stdout' for standard output,
   'f' for file output in directory "data"
   -filename   output filename (default: 'bui.csv')
   -id   ID of a group of genes or a single gene (default: 'All')
   -translate   1 when translates using standard codon table (default: 0)
   -del_key   regular expression to delete key (i.e. amino acids and nucleotides)
   (default: '[^ACDEFGHIKLMNPQRSTVWYacgtU]')
   -position   codon position (default: '');
   '' when assessing overall base usage of the gene
   '1' when assessing base usage at 1st position of codons
   '2' when assessing base usage at 2nd position of codons
   '3' when assessing base 
</p>

<p>
G-language SOAP service is provided by the Institute for Advanced Biosciences, Keio University.<br />

The original web service is located at the following URL:<br />
http://www.g-language.org/wiki/soap
</p>

<p>
WSDL(RPC/Encoded) file is located at:<br />
http://soap.g-language.org/g-language.wsdl
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with gbui

% gbui NC_000913.gbk
NC_000913
Retrieval successful


</pre></td></tr></table>



<h2>Command line argumants</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-translate</td>
<td>boolean</td>
<td>Include when translating using standard codon table</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-position</td>
<td>string</td>
<td>Codon position</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-id</td>
<td>string</td>
<td>ID of a group of genes or a single gene</td>
<td>Any string</td>
<td>All</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-delkey</td>
<td>string</td>
<td>Regular expression to delete key</td>
<td>Any string</td>
<td>[^ACDEFGHIKLMNPQRSTVWYacgtU]</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-accid</td>
<td>boolean</td>
<td>Include to use sequence accession ID as query</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
gbui reads nucleotide sequences.
</p>

<h2>Ouptput file format</h2>

<p>
The output from gbui is to raw CSV format.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>

   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Wev Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>
<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa (gaou@sfc.keio.ac.jp)
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

    None.

