<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gcgr </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emb
oss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gcgr
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Create a Chaos Game Representation of a given sequence
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   This program creates a Chaos Game Representation (CGR) image of a given<br>
   sequence.<br>
<br>
   CGR is generated by the following procedure:<br>
   1. Start from position (0,0) or the origin of two dimensional coordinate.<br>
   Four nucleotides are located at the four corners: <br>
   A: (-1, 1)  upper left<br>
   T: (1, -1)  lower right<br>
   G: (1, 1)   upper right<br>
   C: (-1, -1) lower left<br>
   2. For each nucleotide, move and mark the new location which is halfway<br>
   between the current location and the nucleotide. <br>
   For example, if the first letter is A, position is moved from (0,0) to <br>
   midpoint between (-1, 1) and (0,0), which is (-0.5, 0.5).<br>
   3. Repeat this procedure for all nucleotides.<br>
<br>
   CGR is a generalized scale-independent Markov probability table for the<br>
   sequence, and oligomer tables (see document for "gkmer_table" function) can<br>
   be deduced from CGR image.<br>
<br>
<br>
   More detailed documentation is available at <br>
   http://www.g-language.org/wiki/cgr<br>
 <br>
  <br>
   G-language SOAP service is provided by the<br>
   Institute for Advanced Biosciences, Keio University.<br>
   The original web service is located at the following URL:<br>
<br>
   http://www.g-language.org/wiki/soap<br>
<br>
   WSDL(RPC/Encoded) file is located at:<br>
<br>
   http://soap.g-language.org/g-language.wsdl<br>
<br>
   Documentation on G-language Genome Analysis Environment methods are<br>
   provided at the Document Center<br>
<br>
   http://ws.g-language.org/gdoc/
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with gcgr

% gcgr NC_000913.gbk
Create a Chaos Game Representation of a given sequence
Output filename [gcgr.1.png]:

   Creates gcgr.1.png

</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-filename</td>
<td>string</td>
<td>(no help text) string value</td>
<td>Any string</td>
<td>gcgr.png</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-width</td>
<td>integer</td>
<td>Width of image</td>
<td>Any integer value</td>
<td>1024</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-level</td>
<td>integer</td>
<td>Zoom level for Google Maps representation</td>
<td>Any integer value</td>
<td>5</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-accid</td>
<td>string</td>
<td>Include to use sequence accession ID as query</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-show</td>
<td>toggle</td>
<td>Include to show png with X11 window</td>
<td>Toggle value Yes/No</td>
<td>No</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   gcgr reads nucleotide sequences.
</p>

<h2>Ouptut file format</h2>

<p>
   Creates gcgr.1.png

Command line arguments

   Standard (Mandatory) qualifiers (* if not always prompted):
  [-sequence]          seqall     Nucleotide sequence(s) filename and optional
                                  format, or reference (input USA)
*  -filename           string     [gcgr.png] (no help text) string value (Any
                                  string)

   Additional (Optional) qualifiers: (none)
   Advanced (Unprompted) qualifiers:
   -width              integer    [1024] Width of image (Any integer value)
   -level              integer    [5] Zoom level for Google Maps
                                  representation (Any integer value)
   -accid              string     Include to use sequence accession ID as
                                  query (Any string)
   -show               toggle     [N] Include to show png with X11 window

   General qualifiers:
   -help               boolean    Report command line options and exit. More
                                  information on associated and general
                                  qualifiers can be found with -help -verbose


Input file format

   gcgr reads nucleotide sequences.

Output file format

   The output from gcgr is to png file.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
   Arakawa K, Oshita K, Tomita M (2009) "A web server for interactive and
      zoomable Chaos Game Representation images", Source Code for Biol Med.,
      4:6.

   Almeida JS, Carrico JA, Maretzek A, Noble PA, Fletcher M (2001) 
      "Analysis of genomic sequences by Chaos Game Representation." 
      Bioinformatics. 17(5):429-437.
      
   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Wev Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>

<h2>Warnings</h2>

<p>
   None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

<tr>
<td><a href="gkmer_table.html">gkmer_table</a></td>
<td>Create an image showing all k-mer abundance within a sequence</td>
</tr>

<tr>
<td><a href="gseq2png.html">gseq2png</a></td>
<td>Converts a sequence to PNG image</td>
</tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa(gaou@g-language.org) 
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

