<!--START OF HEADER - DON'T ALTER --><HTML><HEAD>  <TITLE> EMBOSS: gcodoncompiler </TITLE></HEAD><BODY BGCOLOR="#FFFFFF" text="#000000"><table align=center border=0 cellspacing=0 cellpadding=0><tr><td valign=top><A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="http://soap.g-language.org/gembassy/emboss_explorer/manual/emboss_icon.jpg" alt="" width=150 height=48></a></td><td align=left valign=middle><b><font size="+6">gcodoncompiler</font></b></td></tr></table><br>&nbsp;<p><!--END OF HEADER--><H2> Function </H2>   Calculate various kinds of amino acid and codon usage data<!--DON'T WRITE ANYTHING HERE.IT IS DONE FOR YOU.--><H2>Description</H2><p>   gcodoncompiler calculates various kinds of amino acid and codon usage data.<br />   The following values are calculable:<br />      A0: Absolute amino acid frequency<br />      A1: Relative amino acid frequency<br />      C0: Absolute codon frequency<br />      C1: Relative codon frequency in a complete sequence<br />      C2: Relative codon frequency in each amino acid<br />      C3: Relative synonymous codon usage<br />      C4: Relative adaptiveness<br />      C5: Maximum or minor codon<br />   For amino acids unpresent in a gene, C2-C3 does not calculate the values.<br />   By using R* in place, such values are hypothesized that alternative<br />   synonymous codons are used with equal frequency.<br />    <br />   G-language SOAP service is provided by the<br />   Institute for Advanced Biosciences, Keio University.<br />   The original web service is located at the following URL:<br /><br />   http://www.g-language.org/wiki/soap<br /><br />   WSDL(RPC/Encoded) file is located at:<br /><br />   http://soap.g-language.org/g-language.wsdl<br /><br />   Documentation on G-language Genome Analysis Environment methods are<br />   provided at the Document Center<br /><br />   http://ws.g-language.org/gdoc/<br /><br /></p><H2>Usage</H2>Here is a sample session with gcodoncompiler<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>% gcodoncompiler refseqn:NC_000913Calculate various kinds of amino acid and codon usage dataCodon usage output file [nc_000913.gcodoncompiler]: </pre></td></tr></table>Go to the <a href="#input">input files</a> for this example<br>Go to the <a href="#output">output files</a> for this example<br><br><h2>Command line arguments</h2><table border cellspacing=0 cellpadding=3 bgcolor="#ccccff"><tr bgcolor="#FFFFCC"><th align="left">Qualifier</th><th align="left">Type</th><th align="left">Description</th><th align="left">Allowed values</th><th align="left">Default</th></tr><tr bgcolor="#FFFFCC"><th align="left" colspan=5>Standard (Mandatory) qualifiers</th></tr><tr bgcolor="#FFFFCC"><td>[-sequence]<br>(Parameter 1)</td><td>seqall</td><td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td><td>Readable sequence(s)</td><td><b>Required</b></td></tr><tr bgcolor="#FFFFCC"><td>[-outfile]<br>(Parameter 2)</td><td>outfile</td><td>Codon usage output file</td><td>Output file</td><td><i>&lt;*&gt;</i>.gcodoncompiler</td></tr><tr bgcolor="#FFFFCC"><th align="left" colspan=5>Additional (Optional) qualifiers</th></tr><tr><td colspan=5>(none)</td></tr><tr bgcolor="#FFFFCC"><th align="left" colspan=5>Advanced (Unprompted) qualifiers</th></tr><tr bgcolor="#FFFFCC"><td>-translate</td><td>boolean</td><td>Include to translate using standard codon table</td><td>Boolean value Yes/No</td><td>No</td></tr><tr bgcolor="#FFFFCC"><td>-startcodon</td><td>boolean</td><td>Include to include start codon</td><td>Boolean value Yes/No</td><td>No</td></tr><tr bgcolor="#FFFFCC"><td>-stopcodon</td><td>boolean</td><td>Include to include stop codon</td><td>Boolean value Yes/No</td><td>No</td></tr><tr bgcolor="#FFFFCC"><td>-delkey</td><td>string</td><td>Regular expression to delete key (i.e. amino acids and nucleotides)</td><td>Any string</td><td>[^ACDEFGHIKLMNPQRSTVWYacgtU]</td></tr><tr bgcolor="#FFFFCC"><td>-data</td><td>list</td><td>Kinds of codon usage data. R* hypothesizes amino acids which are not present in the gene</td><td><table><tr><td>A0</td> <td><i>(Absolute amino acid frequency ('AA'))</i></td></tr><tr><td>A1</td> <td><i>(Relative amino acid frequency ('RAAU'))</i></td></tr><tr><td>C0</td> <td><i>(Absolute codon frequency ('AF'))</i></td></tr><tr><td>C1</td> <td><i>(Relative codon frequency in a complete sequence)</i></td></tr><tr><td>C2</td> <td><i>(Relative codon frequency in each amino acid ('RF'))</i></td></tr><tr><td>C3</td> <td><i>(Relative synonymous codon usage ('RSCU'))</i></td></tr><tr><td>C4</td> <td><i>(Relative adaptiveness)</i></td></tr><tr><td>i.e., ratio to maximum of minor codon ('W') C5</td> <td><i>(Maximum (1) or minor (0) codon)</i></td></tr><tr><td>R0</td> <td><i>(Absolute codon frequency ('AF'))</i></td></tr><tr><td>R1</td> <td><i>(Relative codon frequency in a complete sequence)</i></td></tr><tr><td>R2</td> <td><i>(Relative codon frequency in each amino acid ('RF'))</i></td></tr><tr><td>R3</td> <td><i>(Relative synonymous codon usage ('RSCU'))</i></td></tr><tr><td>R4</td> <td><i>(Relative adaptiveness)</i></td></tr><tr><td>i.e., ratio to maximum of minor codon ('W') R5</td> <td><i>(Maximum (1) or minor (0) codon)</i></td></tr></table></td><td>R0</td></tr><tr bgcolor="#FFFFCC"><td>-[no]accid</td><td>boolean</td><td>Include to use sequence accession ID as query</td><td>Boolean value Yes/No</td><td>Yes</td></tr></table><h2 id="input">Input file format</h2><p>   The database definitions for following commands are available at<br />   http://soap.g-language.org/kbws/embossrc<br /><br />   gcodoncompiler reads one or more nucleotide sequences.<br /><br /></p><h2 id="output">Output file format</h2><p>   The output from gcodoncompiler is to a plain text file.<br /><br />   File: nc_000913.gcodoncompiler<br /><br /><table width="90%"><tr><td bgcolor="#CCFFCC">Sequence: NC_000913<br />Agca,Agcc,Agcg,Agct,Ctgc,Ctgt,Dgac,Dgat,Egaa,Egag,Fttc,Fttt,Ggga,Gggc,Gggg,Gggt,Hcac,Hcat,Iata,Iatc,Iatt,Kaaa,Kaag,Lcta,Lctc,Lctg,Lctt,Ltta,Lttg,Matg,Naac,Naat,Pcca,Pccc,Pccg,Pcct,Qcaa,Qcag,Raga,Ragg,Rcga,Rcgc,Rcgg,Rcgt,Sagc,Sagt,Stca,Stcc,Stcg,Stct,Taca,Tacc,Tacg,Tact,Utga,Vgta,Vgtc,Vgtg,Vgtt,Wtgg,Ytac,Ytat,locus_tag<br />26551,33911,44924,20010,8486,6707,25234,42161,52362,23474,21841,29334,10226,39395,14472,32678,12830,16952,5356,33359,40221,44272,13398,5079,14709,70441,14410,18097,17936,32971,28329,22786,11063,7142,30994,9130,20216,38169,2495,1366,4529,29308,6991,27864,21132,11323,9159,11332,11759,10992,8979,31001,18989,11581,3,14337,20240,34499,24056,20071,16088,21069,<br /></td></tr></table></p><h2>Data files</h2><p>None.</p><h2>Notes</h2><p>None.</p><h2>References</h2><pre>   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.   Arakawa, K. and Tomita, M. (2006) G-language System as a Platform for      large-scale analysis of high-throughput omics data, J. Pest Sci.,      31, 7.   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome      Analysis Environment with REST and SOAP Web Service Interfaces,      Nucleic Acids Res., 38, W700-W705.</pre><h2>Warnings</h2><p>None.</p><h2>Diagnostic Error Messages</h2><p>None.</p><h2>Exit status</h2><p>It always exits with a status of 0.</p><h2>Known bugs</h2><p>None.</p><h2>See also</h2><table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th><th>Description</th></tr><tr><td><a href="gaminoinfo.html">gaminoinfo</a></td><td>Prints out basic amino acid sequence statistics</td></tr><tr><td><a href="gaaui.html">gaaui</a></td><td>Calculates various indece of amino acid usage</td></tr></table><h2>Author(s)</h2><pre>   Hidetoshi Itaya (celery@g-language.org)   Institute for Advanced Biosciences, Keio University   252-0882 Japan  Kazuharu Arakawa (gaou@sfc.keio.ac.jp)  Institute for Advanced Biosciences, Keio University  252-0882 Japan</pre><h2>History</h2>   2012 - Written by Hidetoshi Itaya<h2>Target users</h2>   This program is intended to be used by everyone and everything, from   naive users to embedded scrips.<h2>Comments</h2>   None.</BODY></HTML>