<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gconsensusz </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="http://soap.g-language.org/gembassy/emboss_explorer/manual/emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gconsensusz
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Calculate consensus in given array of sequences
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   gconsensusz calculates the consensus of given list of sequences, using<br />
   Z-score. The Z-score will show higher values when the sequences are biased<br />
   to a single character within the list.<br />
    <br />
   G-language SOAP service is provided by the<br />
   Institute for Advanced Biosciences, Keio University.<br />
   The original web service is located at the following URL:<br />
<br />
   http://www.g-language.org/wiki/soap<br />
<br />
   WSDL(RPC/Encoded) file is located at:<br />
<br />
   http://soap.g-language.org/g-language.wsdl<br />
<br />
   Documentation on G-language Genome Analysis Environment methods are<br />
   provided at the Document Center<br />
<br />
   http://ws.g-language.org/gdoc/<br />
<br />

</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

Here is a sample session with gconsensusz

% gconsensusz consensus.fasta
Calculate consensus in given array of sequences
Program compseq output file (optional) [rs_000000.gconsensusz]: 

   Go to the input files for this example
   Go to the output files for this example

   Example 2

% gconsensusz consensus.fasta -plot -graph png
Calculate consensus in given array of sequences
Created gconsensusz.1.png

   Go to the input files for this example
   Go to the output files for this example



</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-graph</td>
<td>xygraph</td>
<td>Graph type</td>
<td>EMBOSS has a list of known devices, including ps, hpgl, hp7470, hp7580, meta, cps, x11, tek, tekt, none, data, xterm, png, gif, svg</td>
<td><i>EMBOSS_GRAPHICS</i> value, or x11</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-outfile</td>
<td>outfile</td>
<td>Program compseq output file (optional)</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.gconsensusz</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-high</td>
<td>integer</td>
<td>Z value greater than which is significant</td>
<td>Any integer value</td>
<td>1</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-low</td>
<td>float</td>
<td>Z value less than which is insignificant</td>
<td>Any numeric value</td>
<td>0.2</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-plot</td>
<td>toggle</td>
<td>Include to plot result</td>
<td>Toggle value Yes/No</td>
<td>No</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   The database definitions for following commands are available at<br />
   http://soap.g-language.org/kbws/embossrc<br />
<br />
   gconsensusz reads one or more nucleotide sequences.<br />
<br />

</p>

<h2>Output file format</h2>

<p>
   The output from gconsensusz is to a plain text file or the EMBOSS graphics device.<br />
<br />
   File: rs_000000.gconsensusz<br />
<br />
0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,4.26701934177052,1.74762197406504,0.487923290212298,3.00732065791778,0.487923290212298,0.487923290212298,0.487923290212298,0.487923290212298,3.00732065791778,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,0.487923290212298,3.00732065791778,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,3.00732065791778,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,3.00732065791778,0.487923290212298,3.00732065791778,1.74762197406504,3.00732065791778,0.487923290212298,1.74762197406504,1.74762197406504,3.00732065791778,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,3.00732065791778,5.52671802562326,3.00732065791778,3.00732065791778,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,3.00732065791778,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,3.00732065791778,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,3.00732065791778,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,3.00732065791778,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,3.00732065791778,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,3.00732065791778,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,3.00732065791778,1.74762197406504,3.00732065791778,1.74762197406504,0.487923290212298,3.00732065791778,3.00732065791778,3.00732065791778,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,3.00732065791778,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,4.26701934177052,0.487923290212298,3.00732065791778,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,3.00732065791778,3.00732065791778,4.26701934177052,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,3.00732065791778,1.74762197406504,3.00732065791778,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,3.00732065791778,3.00732065791778,1.74762197406504,1.74762197406504,1.74762197406504,3.00732065791778,1.74762197406504,3.00732065791778,1.74762197406504,1.74762197406504,1.74762197406504,4.26701934177052,0.487923290212298,3.00732065791778,1.74762197406504,0.487923290212298,1.74762197406504,3.00732065791778,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,3.00732065791778,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,1.74762197406504,3.00732065791778,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,3.00732065791778,0.487923290212298,0.487923290212298,1.74762197406504,3.00732065791778,0.487923290212298,1.74762197406504,0.487923290212298,3.00732065791778,1.74762197406504,3.00732065791778,0.487923290212298,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,3.00732065791778,0.487923290212298,3.00732065791778,3.00732065791778,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,0.487923290212298,3.00732065791778,0.487923290212298,3.00732065791778,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,4.26701934177052,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,3.00732065791778,1.74762197406504,4.26701934177052,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,3.00732065791778,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,3.00732065791778,1.74762197406504,1.74762197406504,3.00732065791778,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,5.52671802562326,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,3.00732065791778,1.74762197406504,0.487923290212298,3.00732065791778,1.74762197406504,3.00732065791778,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,3.00732065791778,1.74762197406504,1.74762197406504,0.487923290212298,0.487923290212298,1.74762197406504,1.74762197406504,3.00732065791778,0.487923290212298,1.74762197406504,1.74762197406504,3.00732065791778,0.487923290212298,1.74762197406504,1.74762197406504,0.487923290212298,1.74762197406504,0.487923290212298,<br />
<br />
<br />

</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>References</h2>

<pre>
   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platform for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Web Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.

</pre>

<h2>Warnings</h2>

<p>
None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

<tr>
<td><a href="gdist_in_cc.html">gdist_in_cc</a></td>
<td>Calculates the distance between two loci in circular chromosomes</td>
</tr><tr>
<td><a href="gpalindrome.html">gpalindrome</a></td>
<td>Searches palindrome sequences</td>
</tr><tr>
<td><a href="gseqinfo.html">gseqinfo</a></td>
<td>Prints out basic nucleotide sequence statistics</td>
</tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-0882 Japan

  Kazuharu Arakawa (gaou@sfc.keio.ac.jp)
  Institute for Advanced Biosciences, Keio University
  252-0882 Japan</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

