<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gdelta_gcskew </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emb
oss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gdelta_gcskew
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Calculate strand bias of bacterial genome using delta GC skew index
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   This method calculate strand bias of bacterial genome using delta GC skew index,<br>
   first proposed by Rocha et al. (2001) , and further extended<br>
   in Rocha et al. (2006). Basic idea of delta GC skew index is to calculate the <br>
   difference of GC skew in coding regions residing in leading and lagging strands.<br>
   Rocha et al. (2001) calculates delta GC skew index using the third codon position of genes,<br>
   and Rocha et al. (2006) modified to only use >fourfold degenerate codons.<br>
<br>
 <br>
   G-language SOAP service is provided by the<br>
   Institute for Advanced Biosciences, Keio University.<br>
   The original web service is located at the following URL:<br>
<br>
   http://www.g-language.org/wiki/soap<br>
<br>
   WSDL(RPC/Encoded) file is located at:<br>
<br>
   http://soap.g-language.org/g-language.wsdl<br>
<br>
   Documentation on G-language Genome Analysis Environment methods are<br>
   provided at the Document Center<br>
<br>
   http://ws.g-language.org/gdoc/
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with gdelta_gcskew

% gdelta_gcskew NC_000913.gbk
Calculate strand bias of bacterial genome using delta GC skew index

</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-at</td>
<td>boolean</td>
<td>'1' when observing AT skew instead of GC skew</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-purine</td>
<td>boolean</td>
<td>'1' when observing purine (AG/TC) skew</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-keto</td>
<td>boolean</td>
<td>'1' when observing keto (TG/AC) skew</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-method</td>
<td>string</td>
<td>Choose the nucleotides to use 'degenerate', 'gc3', or 'all'</td>
<td>Any string</td>
<td>degenerate</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-accid</td>
<td>string</td>
<td>Include to use sequence accession ID as query</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   gdelta_gcskew reads nucleotide sequences.
</p>

<h2>Ouptut file format</h2>

<p>
   The output from gdelta_gcskew is to standard output.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
   Rocha EPC et al. (2001) "Ongoing Evolution of Strand Composition in Bacterial Genomes",
      Molecular Biology and Evolution, 18(9):1789-1799

   Rocha EPC et al. (2006) "Similar compositional biases are caused by very different
      mutational effects", Genome Research, 16:1537-1547

   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Wev Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>

<h2>Warnings</h2>

<p>
   None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

<tr>
<td><a href="NA.html">NA</a></td>
<td>appdescription</td>
</tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa(gaou@g-language.org) 
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

