<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: genret </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="http://www.g-language.org/gembassy/emboss_explorer/manual/emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
genret
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Retrieves various gene related information from genome flatfile
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   genret reads in one or more genome flatfiles and retrieves various data from
   the input file. It is a wrapper program to the G-language REST service,
   where a method is specified by giving a string to the "method" qualifier. By
   default, genret will parse the input file to retrieve the accession ID
   (or name) of the genome to query G-language REST service. By setting the
   "accid" qualifier to false (or 0), genret will instead parse the sequence
   and features of the genome to create a GenBank formatted flatfile and upload
   the file to the G-language web server. Using the file uploaded, genret will
   execute the method provided.<br>
<br>
   genret is able to perform a variety of tasks, incluing the retrieval of
   sequence upstream, downstream, or around the start or stop codon,
   translated gene sequences search of gene data by keyword.<br>
<br>
   Details on G-language REST service is available from the wiki page<br>
<br>
   http://www.g-language.org/wiki/rest<br>
<br>
   Documentation on G-language Genome Analysis Environment methods are<br>
   provided at the Document Center<br>
<br>
   http://ws.g-language.org/gdoc/
</p>

<H2>Usage</H2>

   Here is a sample session with <b>genret</b><br><br>
   Retrieving sequences upstream, downstream, or around the start/stop codons.
   <br>
   The following example shows the retrieval of sequence around the start
   codons of all genes.<br><br>
   Available methods are:<br>
      after_startcodon<br>
      after_stopcodon<br>
      around_startcodon<br>
      around_stopcodon<br>
      before_startcodon<br>
      before_stopcodon<br><br>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>
% genret
Retrieves various gene related information from genome flatfile
Input nucleotide sequence(s): refseqn:NC_000913
Gene name(s) to lookup [*]:
Feature to access: around_startcodon
Full text output file [nc_000913.around_startcodon]:
</pre></td></tr></table><br>

   <b>Example 2</b><br><br>

   Retrieving translations of coding sequences.<br>
   The following example will retrieve the translated protein sequence of
   the "recA" gene.<br><br>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>
% genret
Retrieves various gene related information from genome flatfile
Input nucleotide sequence(s): refseqn:NC_000913
Gene name(s) to lookup [*]: recA
Feature to access: translation
Full text output file [nc_000913.translation]: stdout
>recA
MAIDENKQKALAAALGQIEKQFGKGSIMRLGEDRSMDVETISTGSLSLDIALGAGGLPMGR
IVEIYGPESSGKTTLTLQVIAAAQREGKTCAFIDAEHALDPIYARKLGVDIDNLLCSQPDT
GEQALEICDALARSGAVDVIVVDSVAALTPKAEIEGEIGDSHMGLAARMMSQAMRKLAGNL
KQSNTLLIFINQIRMKIGVMFGNPETTTGGNALKFYASVRLDIRRIGAVKEGENVVGSETR
VKVVKNKIAAPFKQAEFQILYGEGINFYGELVDLGVKEKLIEKAGAWYSYKGEKIGQGKAN
ATAWLKDNPETAKEIEKKVRELLLSNPNSTPDFSVDDSEGVAETNEDF
</pre></td></tr></table><br>

   <b>Example 3</b><br><br>

   Retrieving feature information of the genes.<br>
   The following example shows the retrieval of start positions for each gene.
   The values for the keys in GenBank format is available for retrieval.
   (ex. start end direction GO* etc.)<br><br>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>
% genret
Retrieves various gene related information from genome flatfile
Input nucleotide sequence(s): refseqn:NC_000913
Gene name(s) to lookup [*]:
Feature to access: start
Full text output file [nc_000913.start]:
</pre></td></tr></table><br>

   <b>Example 4</b><br><br>

   Using a file with a list of gene names.<br>
   The following example will retrieve the strand direction for each gene
   listed in the "gene_list.txt" file. String prefixed with an "@" or "list::"
   will be interpreted as file names.<br><br>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>
% genret
Retrieves various gene features from genome flatfile
Input nucleotide sequence(s): refseqn:NC_000913
List of gene name(s) to report [*]: @gene_list.txt
Name of gene feature to access: direction
Full text output file [nc_000913.direction]: stdout
gene,direction
thrA,direct
thrB,direct
thrC,direct
</pre></td></tr></table><br>

   <b>Example 5</b><br><br>

   Passing extra arguments to the methods.<br>
   The following example shows the retrieval of 30 base pairs around the
   start codon of the "recA" gene. By default, the "around_startcodon" method
   returns 200 base pairs around the start codon. Using the "-argument"
   qualifier allows the user to change this value.<br><br>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>
% genret refseqn:NC_000913 recA around_startcodon -argument 30,30 stdout
Retrieves various gene features from genome flatfile
>recA
ccggtattacccggcatgacaggagtaaaaatggctatcgacgaaaacaaacagaaagcgt
tg
</pre></td></tr></table><br>

<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-gene]<br>(Parameter 2)</td>
<td>string</td>
<td>Gene name(s) to lookup</td>
<td>Any string</td>
<td>*</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-access]<br>(Parameter 3)</td>
<td>string</td>
<td>Feature to access</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-outfile]<br>(Parameter 4)</td>
<td>outfile</td>
<td>Full text output file</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.genret</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-argument</td>
<td>string</td>
<td>Option to give to method</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-[no]accid</td>
<td>boolean</td>
<td>Include to use sequence accession ID as query</td>
<td>Boolean value Yes/No</td>
<td>Yes</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   Database definitions for the examples are included in the embossrc_template
   file of the Keio Bioinformatcs Web Service (KBWS) package.<br><br>

   Input files for usage example 4<br><br>

   File: gene_list.txt<br><br>
<pre>
thrA
thrB
thrC
</pre>
</p>

<h2>Output file format</h2>

<p>
   Output files for usage example 1<br><br>

   File: nc_000913.around_startcodon<br><br>

<pre>
>thrL
cgtgagtaaattaaaattttattgacttaggtcactaaatactttaaccaatataggcata
gcgcacagacagataaaaattacagagtacacaacatccatgaaacgcattagcaccacca
ttaccaccaccatcaccattaccacaggtaacggtgcgggctgacgcgtacaggaaacaca
gaaaaaagcccgcacctgac
>thrA
aggtaacggtgcgggctgacgcgtacaggaaacacagaaaaaagcccgcacctgacagtgc
gggctttttttttcgaccaaaggtaacgaggtaacaaccatgcgagtgttgaagttcggcg
gtacatcagtggcaaatgcagaacgttttctgcgtgttgccgatattctggaaagcaatgc
caggcaggggcaggtggcca

   [Part of this file has been deleted for brevity]

>yjjY
tgcatgtttgctacctaaattgccaactaaatcgaaacaggaagtacaaaagtccctgacc
tgcctgatgcatgctgcaaattaacatgatcggcgtaacatgactaaagtacgtaattgcg
ttcttgatgcactttccatcaacgtcaacaacatcattagcttggtcgtgggtactttccc
tcaggacccgacagtgtcaa
>yjtD
tttttctgcgacttacgttaagaatttgtaaattcgcaccgcgtaataagttgacagtgat
cacccggttcgcggttatttgatcaagaagagtggcaatatgcgtataacgattattctgg
tcgcacccgccagagcagaaaatattggggcagcggcgcgggcaatgaaaacgatggggtt
tagcgatctgcggattgtcg
</pre>
</p>

<p>
   Output files for usage example 1<br><br>

   File: nc_000913.start<br><br>

<pre>
gene,start
thrL,190
thrA,337
thrB,2801
thrC,3734
yaaX,5234
yaaA,5683
yaaJ,6529
talB,8238
mog,9306

   [Part of this file has been deleted for brevity]

yjjX,4631256
ytjC,4631820
rob,4632464
creA,4633544
creB,4634030
creC,4634719
creD,4636201
arcA,4637613
yjjY,4638425
yjtD,4638965
</pre>
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platform for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Web Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>

<h2>Warnings</h2>

<p>
   None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa(gaou@g-language.org) 
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

