<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
  <head>
    <title>EMBOSS: genret manual</title>
    <link rel='stylesheet' type='text/css' href='/gembassy/emboss_explorer/style/emboss.css' />
    
  </head>
  <body>
    <div id='manual'>
      <!-- tfm output starts here -->
      <table align=center border=0 cellspacing=0 cellpadding=0><tr><td valign=top><A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="/gembassy/emboss_explorer/manual/emboss_icon.jpg" alt="" width=150 height=48></a></td><td align=left valign=middle><b><font size="+6">genret</font></b></td></tr></table><br>&nbsp;<p><!--END OF HEADER--><H2> Function </H2>   Retrieves various gene related information from genome flatfile<!--DON'T WRITE ANYTHING HERE.IT IS DONE FOR YOU.--><H2>Description</H2><p>   genret reads in one or more genome flatfiles and retrieves various data from<br />   the input file. It is a wrapper program to the G-language REST service,<br />   where a method is specified by giving a string to the "method" qualifier. By<br />   default, genret will parse the input file to retrieve the accession ID<br />   (or name) of the genome to query G-language REST service. By setting the<br />   "accid" qualifier to false (or 0), genret will instead parse the sequence<br />   and features of the genome to create a GenBank formatted flatfile and upload<br />   the file to the G-language web server. Using the file uploaded, genret will<br />   execute the method provided.<br /><br />   genret is able to perform a variety of tasks, incluing the retrieval of<br />   sequence upstream, downstream, or around the start or stop codon,<br />   translated gene sequences search of gene data by keyword.<br /><br />   Details on G-language REST service is available from the wiki page<br /><br />   http://www.g-language.org/wiki/rest<br /><br />   Documentation on G-language Genome Analysis Environment methods are<br />   provided at the Document Center<br /><br />   http://ws.g-language.org/gdoc/<br /><br /></p><H2>Usage</H2>Here is a sample session with genret<br><br>Retrieving sequences upstream, downstream, or around the start/stop codons.The following example shows the retrieval of sequence around the startcodons of all genes.<br><br>Genes to access are specified by regular expression. '*' stands for everygene.<br><br>Available methods are:<br>   after_startcodon<br>   after_stopcodon<br>   around_startcodon<br>   around_stopcodon<br>   before_startcodon<br>   before_stopcodon<br><br><table width="90%"><tr><td bgcolor="#CCFFFF"><pre>% genretRetrieves various gene related information from genome flatfileInput nucleotide sequence(s): refseqn:NC_000913Gene name(s) to lookup [*]:Feature to access: around_startcodonFull text output file [nc_000913.around_startcodon]:</pre></td></tr></table>Go to the <a href="#input">input files</a> for this example<br>Go to the <a href="#output">output files</a> for this example<br><br>Example 2<br><br>Using flat text as target genes. The names can be split with with a space, comma, or vertical bar.<br><br><table width="90%"><tr><td bgcolor="#CCFFFF"><pre>% genretRetrieves various gene related information from genome flatfileInput nucleotide sequence(s): refseqn:NC_000913List of gene name(s) to report [*]: recA,recBName of gene feature to access: translationSequence output file [nc_000913.translation.genret]: stdout>recAMAIDENKQKALAAALGQIEKQFGKGSIMRLGEDRSMDVETISTGSLSLDIALGAGGLPMGRIVEIYGPESSGKTTLTLQVIAAAQREGKTCAFIDAEHALDPIYARKLGVDIDNLLCSQPDTGEQALEICDALARSGAVDVIVVDSVAALTPKAEIEGEIGDSHMGLAARMMSQAMRKLAGNLKQSNTLLIFINQIRMKIGVMFGNPETTTGGNALKFYASVRLDIRRIGAVKEGENVVGSETRVKVVKNKIAAPFKQAEFQILYGEGINFYGELVDLGVKEKLIEKAGAWYSYKGEKIGQGKANATAWLKDNPETAKEIEKKVRELLLSNPNSTPDFSVDDSEGVAETNEDF>recBMSDVAETLDPLRLPLQGERLIEASAGTGKTFTIAALYLRLLLGLGGSAAFPRPLTVEELLVVTFTEAATAELRGRIRSNIHELRIACLRETTDNPLYERLLEEIDDKAQAAQWLLLAERQMDEAAVFTIHGFCQRMLNLNAFESGMLFEQQLIEDESLLRYQACADFWRRHCYPLPREIAQVVFETWKGPQALLRDINRYLQGEAPVIKAPPPDDETLASRHAQIVARIDTVKQQWRDAVGELDALIESSGIDRRKFNRSNQAKWIDKISAWAEEETNSYQLPESLEKFSQRFLEDRTKAGGETPRHPLFEAIDQLLAEPLSIRDLVITRALAEIRETVAREKRRRGELGFDDMLSRLDSALRSESGEVLAAAIRTRFPVAMIDEFQDTDPQQYRIFRRIWHHQPETALLLIGDPKQAIYAFRGADIFTYMKARSEVHAHYTLDTNWRSAPGMVNSVNKLFSQTDDAFMFREIPFIPVKSAGKNQALRFVFKGETQPAMKMWLMEGESCGVGDYQSTMAQVCAAQIRDWLQAGQRGEALLMNGDDARPVRASDISVLVRSRQEAAQVRDALTLLEIPSVYLSNRDSVFETLEAQEMLWLLQAVMTPERENTLRSALATSMMGLNALDIETLNNDEHAWDVVVEEFDGYRQIWRKRGVMPMLRALMSARNIAENLLATAGGERRLTDILHISELLQEAGTQLESEHALVRWLSQHILEPDSNASSQQMRLESDKHLVQIVTIHKSKGLEYPLVWLPFITNFRVQEQAFYHDRHSFEAVLDLNAAPESVDLAEAERLAEDLRLLYVALTRSVWHCSLGVAPLVRRRGDKKGDTDVHQSALGRLLQKGEPQDAAGLRTCIEALCDDDIAWQTAQTGDNQPWQVNDVSTAELNAKTLQRLPGDNWRVTSYSGLQQRGHGIAQDLMPRLDVDAAGVASVVEEPTLTPHQFPRGASPGTFLHSLFEDLDFTQPVDPNWVREKLELGGFESQWEPVLTEWITAVLQAPLNETGVSLSQLSARNKQVEMEFYLPISEPLIASQLDTLIRQFDPLSAGCPPLEFMQVRGMLKGFIDLVFRHEGRYYLLDYKSNWLGEDSSAYTQQAMAAAMQAHRYDLQYQLYTLALHRYLRHRIADYDYEHHFGGVIYLFLRGVDKEHPQQGIYTTRPNAGLIALMDEMFAGMTLEEA</pre></td></tr></table>Go to the <a href="#input">input files</a> for this example<br>Go to the <a href="#output">output files</a> for this example<br><br>Example 3<br><br>Using a file with a list of gene names.The following example will retrieve the strand direction for each genelisted in the "gene_list.txt" file. String prefixed with an "@" or "list::"will be interpreted as file names.<br><br><table width="90%"><tr><td bgcolor="#CCFFFF"><pre>% genretRetrieves various gene features from genome flatfileInput nucleotide sequence(s): refseqn:NC_000913List of gene name(s) to report [*]: @gene_list.txtName of gene feature to access: directionFull text output file [nc_000913.direction]: stdoutgene,directionthrA,directthrB,directthrC,direct</pre></td></tr></table>Example 4<br><br>Retrieving translations of coding sequences.<br>The following example will retrieve the translated protein sequence ofthe "recA" gene.<br><br><table width="90%"><tr><td bgcolor="#CCFFFF"><pre>% genretRetrieves various gene related information from genome flatfileInput nucleotide sequence(s): refseqn:NC_000913Gene name(s) to lookup [*]: recAFeature to access: translationFull text output file [nc_000913.translation]: stdout>recAMAIDENKQKALAAALGQIEKQFGKGSIMRLGEDRSMDVETISTGSLSLDIALGAGGLPMGRIVEIYGPESSGKTTLTLQVIAAAQREGKTCAFIDAEHALDPIYARKLGVDIDNLLCSQPDTGEQALEICDALARSGAVDVIVVDSVAALTPKAEIEGEIGDSHMGLAARMMSQAMRKLAGNLKQSNTLLIFINQIRMKIGVMFGNPETTTGGNALKFYASVRLDIRRIGAVKEGENVVGSETRVKVVKNKIAAPFKQAEFQILYGEGINFYGELVDLGVKEKLIEKAGAWYSYKGEKIGQGKANATAWLKDNPETAKEIEKKVRELLLSNPNSTPDFSVDDSEGVAETNEDF</pre></td></tr></table>Go to the <a href="#input">input files</a> for this example<br>Go to the <a href="#output">output files</a> for this example<br><br>Example 5<br><br>Retrieving feature information of the genes.<br>The following example will retrieve the start positions for each gene.The values for the keys in GenBank format is available for retrieval.(ex. start end direction GO* etc.)<br>Positions will be returned with a 1 start value.<br><br><table width="90%"><tr><td bgcolor="#CCFFFF"><pre>% genretRetrieves various gene related information from genome flatfileInput nucleotide sequence(s): refseqn:NC_000913Gene name(s) to lookup [*]:Feature to access: startFull text output file [nc_000913.start]:</pre></td></tr></table>Go to the <a href="#input">input files</a> for this example<br>Go to the <a href="#output">output files</a> for this example<br><br>Example 6<br><br>Passing extra arguments to the methods.<br>The following example shows the retrieval of 30 base pairs around thestart codon of the "recA" gene. By default, the "around_startcodon" methodreturns 200 base pairs around the start codon. Using the "-argument"qualifier allows the user to change this value.<br><br><table width="90%"><tr><td bgcolor="#CCFFFF"><pre>% genret refseqn:NC_000913 recA around_startcodon -argument 30,30 stdoutRetrieves various gene features from genome flatfile>recAccggtattacccggcatgacaggagtaaaaatggctatcgacgaaaacaaacagaaagcgttg</pre></td></tr></table>Go to the <a href="#input">input files</a> for this example<br>Go to the <a href="#output">output files</a> for this example<br><br>Example 7<br><br>Re-annotating a flatfile.genret supports re-annotation of a genome flatfile via Restauro-Gservice developed by our team.The original software is available at [<a href="http://restauro-g.iab.keio.ac.jp/">http://restauro-g.iab.keio.ac.jp</a>].<br><br><table width="90%"><tr><td bgcolor="#CCFFFF"><pre>% genret refseqn:NC_000913 '*' annotate nc_000913-annotate.gbkRetrieves various gene features from genome flatfile</pre></td></tr></table>Go to the <a href="#input">input files</a> for this example<br>Go to the <a href="#output">output files</a> for this example<br><br><h2>Command line arguments</h2><table border cellspacing=0 cellpadding=3 bgcolor="#ccccff"><tr bgcolor="#FFFFCC"><th align="left">Qualifier</th><th align="left">Type</th><th align="left">Description</th><th align="left">Allowed values</th><th align="left">Default</th></tr><tr bgcolor="#FFFFCC"><th align="left" colspan=5>Standard (Mandatory) qualifiers</th></tr><tr bgcolor="#FFFFCC"><td>[-sequence]<br>(Parameter 1)</td><td>seqall</td><td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td><td>Readable sequence(s)</td><td><b>Required</b></td></tr><tr bgcolor="#FFFFCC"><td>[-gene]<br>(Parameter 2)</td><td>string</td><td>List of gene name(s) to report</td><td>Any string</td><td>*</td></tr><tr bgcolor="#FFFFCC"><td>[-access]<br>(Parameter 3)</td><td>string</td><td>Name of gene feature to access</td><td>Any word</td><td>&nbsp;</td></tr><tr bgcolor="#FFFFCC"><td>[-outfile]<br>(Parameter 4)</td><td>outfile</td><td>Sequence output file</td><td>Output file</td><td><i>&lt;*&gt;</i>.genret</td></tr><tr bgcolor="#FFFFCC"><th align="left" colspan=5>Additional (Optional) qualifiers</th></tr><tr><td colspan=5>(none)</td></tr><tr bgcolor="#FFFFCC"><th align="left" colspan=5>Advanced (Unprompted) qualifiers</th></tr><tr bgcolor="#FFFFCC"><td>-argument</td><td>string</td><td>Extra arguments to pass to method</td><td>Any string</td><td>&nbsp;</td></tr><tr bgcolor="#FFFFCC"><td>-[no]accid</td><td>boolean</td><td>Include to use sequence accession ID as query</td><td>Boolean value Yes/No</td><td>Yes</td></tr></table><h2 id="input">Input file format</h2><p>   Database definitions for the examples are included in the embossrc_template<br />   file of the Keio Bioinformatcs Web Service (KBWS) package.<br /><br />   Input files for usage example 4<br /><br />   File: gene_list.txt<br /><br />thrA<br />thrB<br />thrC<br /><br /></p><h2 id="output">Output file format</h2><p>   Output files for usage example 1<br /><br />   File: nc_000913.around_startcodon<br /><br /><table width="90%"><tr><td bgcolor="#CCFFCC">>thrL<br />cgtgagtaaattaaaattttattgacttaggtcactaaatactttaaccaatataggcata<br />gcgcacagacagataaaaattacagagtacacaacatccatgaaacgcattagcaccacca<br />ttaccaccaccatcaccattaccacaggtaacggtgcgggctgacgcgtacaggaaacaca<br />gaaaaaagcccgcacctgac<br />>thrA<br />aggtaacggtgcgggctgacgcgtacaggaaacacagaaaaaagcccgcacctgacagtgc<br />gggctttttttttcgaccaaaggtaacgaggtaacaaccatgcgagtgttgaagttcggcg<br />gtacatcagtggcaaatgcagaacgttttctgcgtgttgccgatattctggaaagcaatgc<br />caggcaggggcaggtggcca<br /><br /><font color=red>[Part of this file has been deleted for brevity]</font><br /><br />>yjjY<br />tgcatgtttgctacctaaattgccaactaaatcgaaacaggaagtacaaaagtccctgacc<br />tgcctgatgcatgctgcaaattaacatgatcggcgtaacatgactaaagtacgtaattgcg<br />ttcttgatgcactttccatcaacgtcaacaacatcattagcttggtcgtgggtactttccc<br />tcaggacccgacagtgtcaa<br />>yjtD<br />tttttctgcgacttacgttaagaatttgtaaattcgcaccgcgtaataagttgacagtgat<br />cacccggttcgcggttatttgatcaagaagagtggcaatatgcgtataacgattattctgg<br />tcgcacccgccagagcagaaaatattggggcagcggcgcgggcaatgaaaacgatggggtt<br />tagcgatctgcggattgtcg<br /></td></tr></table><br />   Output files for usage example 3<br /><br />   File: nc_000913.start<br /><br /><table width="90%"><tr><td bgcolor="#CCFFCC">gene,start<br />thrL,190<br />thrA,337<br />thrB,2801<br />thrC,3734<br />yaaX,5234<br />yaaA,5683<br />yaaJ,6529<br />talB,8238<br />mog,9306<br /><br /><font color=red>[Part of this file has been deleted for brevity]</font><br /><br />yjjX,4631256<br />ytjC,4631820<br />rob,4632464<br />creA,4633544<br />creB,4634030<br />creC,4634719<br />creD,4636201<br />arcA,4637613<br />yjjY,4638425<br />yjtD,4638965<br /></td></tr></table><br />   Output files for usage example 7<br /><br />   File: ecoli-annotate.gbk<br /><br /><table width="90%"><tr><td bgcolor="#CCFFCC">LOCUS       NC_000913            4639675 bp    DNA     circular BCT 25-OCT-2010<br />DEFINITION  Escherichia coli str. K-12 substr. MG1655 chromosome, complete<br />            genome.<br />ACCESSION   NC_000913<br />VERSION     NC_000913.2  GI:49175990<br />DBLINK      Project: 57779<br />KEYWORDS    .<br />SOURCE      Escherichia coli str. K-12 substr. MG1655<br />  ORGANISM  Escherichia coli str. K-12 substr. MG1655<br />            Bacteria; Proteobacteria; Gammaproteobacteria; Enterobacteriales;<br /><br /><font color="red">[Part of this file has been deleted for brevity]</font><br /><br />     CDS             2801..3733<br />                     /EC_number="2.7.1.39"<br />                     /codon_start="1"<br />                     /db_xref="GI:16127997"<br />                     /db_xref="ASAP:ABE-0000010"<br />                     /db_xref="UniProtKB/Swiss-Prot:P00547"<br />                     /db_xref="ECOCYC:EG10999"<br />                     /db_xref="EcoGene:EG10999"<br />                     /db_xref="GeneID:947498"<br />                     /function="enzyme; Amino acid biosynthesis: Threonine"<br />                     /function="1.5.1.8 metabolism; building block<br />                     biosynthesis; amino acids; threonine"<br />                     /function="7.1 location of gene products; cytoplasm"<br />                     /gene="thrB"<br />                     /gene_synonym="ECK0003; JW0002"<br />                     /locus_tag="b0003"<br />                     /note="GO_component: GO:0005737 - cytoplasm; GO_process:<br />                     GO:0009088 - threonine biosynthetic process"<br />                     /product="homoserine kinase"<br />                     /protein_id="NP_414544.1"<br />                     /rs_com="FUNCTION: Catalyzes the ATP-dependent<br />                     phosphorylation of L- homoserine to L-homoserine<br />                     phosphate (By similarity)."<br />                     /rs_com="CATALYTIC ACTIVITY: ATP + L-homoserine = ADP +<br />                     O-phospho-L- homoserine."<br />                     /rs_com="PATHWAY: Amino-acid biosynthesis; L-threonine<br />                     biosynthesis; L- threonine from L-aspartate: step 4/5."<br />                     /rs_com="SUBCELLULAR LOCATION: Cytoplasm (Potential)."<br />                     /rs_com="SIMILARITY: Belongs to the GHMP kinase family.<br />                     Homoserine kinase subfamily."<br />                     /rs_des="RecName: Full=Homoserine kinase; Short=HK;<br />                     Short=HSK; EC=2.7.1.39;"<br />                     /rs_protein="Level 1: similar to KHSE_ECODH 1.7e-180"<br />                     /rs_xr="EMBL; CP000948; ACB01208.1; -; Genomic_DNA."<br />                     /rs_xr="RefSeq; YP_001728986.1; -."<br />                     /rs_xr="ProteinModelPortal; B1XBC8; -."<br />                     /rs_xr="SMR; B1XBC8; 2-308."<br />                     /rs_xr="EnsemblBacteria; EBESCT00000012034;<br />                     EBESCP00000011562; EBESCG00000011096."<br />                     /rs_xr="GeneID; 6058639; -."<br />                     /rs_xr="GenomeReviews; CP000948_GR; ECDH10B_0003."<br />                     /rs_xr="KEGG; ecd:ECDH10B_0003; -."<br />                     /rs_xr="HOGENOM; HBG646290; -."<br />                     /rs_xr="OMA; GSAHADN; -."<br />                     /rs_xr="ProtClustDB; PRK01212; -."<br />                     /rs_xr="BioCyc; ECOL316385:ECDH10B_0003-MONOMER; -."<br />                     /rs_xr="GO; GO:0005737; C:cytoplasm;<br />                     IEA:UniProtKB-SubCell."<br />                     /rs_xr="GO; GO:0005524; F:ATP binding; IEA:UniProtKB-KW."<br />                     /rs_xr="GO; GO:0004413; F:homoserine kinase activity;<br />                     IEA:EC."<br />                     /rs_xr="GO; GO:0009088; P:threonine biosynthetic process;<br />                     IEA:UniProtKB-KW."<br />                     /rs_xr="HAMAP; MF_00384; Homoser_kinase; 1; -."<br />                     /rs_xr="InterPro; IPR006204; GHMP_kinase."<br />                     /rs_xr="InterPro; IPR013750; GHMP_kinase_C."<br />                     /rs_xr="InterPro; IPR006203; GHMP_knse_ATP-bd_CS."<br />                     /rs_xr="InterPro; IPR000870; Homoserine_kin."<br />                     /rs_xr="InterPro; IPR020568; Ribosomal_S5_D2-typ_fold."<br />                     /rs_xr="InterPro; IPR014721;<br />                     Ribosomal_S5_D2-typ_fold_subgr."<br />                     /rs_xr="Gene3D; G3DSA:3.30.230.10;<br />                     Ribosomal_S5_D2-type_fold; 1."<br />                     /rs_xr="Pfam; PF08544; GHMP_kinases_C; 1."<br />                     /rs_xr="Pfam; PF00288; GHMP_kinases_N; 1."<br />                     /rs_xr="PIRSF; PIRSF000676; Homoser_kin; 1."<br />                     /rs_xr="PRINTS; PR00958; HOMSERKINASE."<br />                     /rs_xr="SUPFAM; SSF54211; Ribosomal_S5_D2-typ_fold; 1."<br />                     /rs_xr="TIGRFAMs; TIGR00191; thrB; 1."<br />                     /rs_xr="PROSITE; PS00627; GHMP_KINASES_ATP; 1."<br />                     /transl_table="11"<br />                     /translation="MVKVYAPASSANMSVGFDVLGAAVTPVDGALLGDVVTVEAAETF<br />                     SLNNLGRFADKLPSEPRENIVYQCWERFCQELGKQIPVAMTLEKNMPIGSGLGSSACS<br />                     VVAALMAMNEHCGKPLNDTRLLALMGELEGRISGSIHYDNVAPCFLGGMQLMIEENDI<br />                     ISQQVPGFDEWLWVLAYPGIKVSTAEARAILPAQYRRQDCIAHGRHLAGFIHACYSRQ<br />                     PELAAKLMKDVIAEPYRERLLPGFRQARQAVAEIGAVASGISGSGPTLFALCDKPETA<br />                     QRVADWLGKNYLQNQEGFVHICRLDTAGARVLEN"<br /><br /><font color="red">[Part of this file has been deleted for brevity]</font><br /><br />  4639201 gcgcagtcgg gcgaaatatc attactacgc cacgccagtt gaactggtgc cgctgttaga<br />  4639261 ggaaaaatct tcatggatga gccatgccgc gctggtgttt ggtcgcgaag attccgggtt<br />  4639321 gactaacgaa gagttagcgt tggctgacgt tcttactggt gtgccgatgg tggcggatta<br />  4639381 tccttcgctc aatctggggc aggcggtgat ggtctattgc tatcaattag caacattaat<br />  4639441 acaacaaccg gcgaaaagtg atgcaacggc agaccaacat caactgcaag ctttacgcga<br />  4639501 acgagccatg acattgctga cgactctggc agtggcagat gacataaaac tggtcgactg<br />  4639561 gttacaacaa cgcctggggc ttttagagca acgagacacg gcaatgttgc accgtttgct<br />  4639621 gcatgatatt gaaaaaaata tcaccaaata aaaaacgcct tagtaagtat ttttc<br />//<br /></td></tr></table></p><h2>Data files</h2><p>None.</p><h2>Notes</h2><p>None.</p><h2>References</h2><pre>   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.   Arakawa, K. and Tomita, M. (2006) G-language System as a Platform for      large-scale analysis of high-throughput omics data, J. Pest Sci.,      31, 7.   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome      Analysis Environment with REST and SOAP Web Service Interfaces,      Nucleic Acids Res., 38, W700-W705.</pre><h2>Warnings</h2><p>None.</p><h2>Diagnostic Error Messages</h2><p>None.</p><h2>Exit status</h2><p>It always exits with a status of 0.</p><h2>Known bugs</h2><p>None.</p><h2>See also</h2><table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th><th>Description</th></tr><tr><td><a href="entret">entret</a></td><td>Retrieve sequence entries from flatfile databases and files</td></tr><tr><td><a href="seqret">seqret</a></td><td>Read and write (return) sequences</td></tr></table><h2>Author(s)</h2><pre>Hidetoshi Itaya (celery@g-language.org)  Institute for Advanced Biosciences, Keio University  252-0882 JapanKazuharu Arakawa (gaou@sfc.keio.ac.jp)  Institute for Advanced Biosciences, Keio University  252-0882 Japan</pre><h2>History</h2>   2012 - Written by Hidetoshi Itaya<h2>Target users</h2>   This program is intended to be used by everyone and everything, from   naive users to embedded scrips.<h2>Comments</h2>   None.</HTML>

      <!-- tfm output ends here -->
    </div>
  </body>
</html>
