<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gentrez </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="http://soap.g-language.org/gembassy/emboss_explorer/manual/emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gentrez
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Search NCBI Entrez in G-language Shell
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   Search NCBI Entrez with keyword through EUtilities. <br>
   This is intended for quick lookup through G-language Shell, <br>
   so only top ten hits are reported.<br>
<br>
  <br>
   G-language SOAP service is provided by the<br>
   Institute for Advanced Biosciences, Keio University.<br>
   The original web service is located at the following URL:<br>
<br>
   http://www.g-language.org/wiki/soap<br>
<br>
   WSDL(RPC/Encoded) file is located at:<br>
<br>
   http://soap.g-language.org/g-language.wsdl<br>
<br>
   Documentation on G-language Genome Analysis Environment methods are<br>
   provided at the Document Center<br>
<br>
   http://ws.g-language.org/gdoc/
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with gentrez

% gentrez -show pubmed 'Venter C'
Search NCBI Entrez in G-language Shell
Output filename [pubmed.Venter C.gentrez]:
    130 entries found in PUBMED: (Showing up to 10 hits)

     1. PMID:   23209482
        Venter C et al. "Food hypersensitivity: diagnosing and managing food allergies and intolerances."
        J Allergy (Cairo)  2012;2012:576017

     2. PMID:   23150738
        Venter C et al. "Nutritional aspects in diagnosis and management of food hypersensitivity-the dietitians role."
        J Allergy (Cairo)  2012;2012:269376

     3. PMID:   23121771
        Papadopoulos NG et al. "Research needs in allergy: an EAACI position paper, in collaboration with EFA."
        Clin Transl Allergy  2012 Nov 2;2(1):21

     4. PMID:   22913718
        Sommer I et al. "Factors influencing food choices of food-allergic consumers: findings from focus groups."
        Allergy  2012 Oct;67(10):1319-22

     5. PMID:   22714410
        Djagaeva I et al. "Three routes to suppression of the neurodegenerative phenotypes caused by kinesin heavy chain mutations."
        Genetics  2012 Sep;192(1):173-83

     6. PMID:   22685976
        Venter C et al. "A topic in 10 questions: weaning the child with cow's milk protein allergy."
        J Fam Health Care  2012 Mar-Apr;22(2):22-3

     7. PMID:   22499144
        Venter C et al. "Guideline fever: an overview of DRACMA, US NIAID and UK NICE guidelines."
        Curr Opin Allergy Clin Immunol  2012 Jun;12(3):302-15

     8. PMID:   22435534
        Meyer R et al. "Practical dietary management of protein energy malnutrition in young children with cow's milk protein allergy."
        Pediatr Allergy Immunol  2012 Jun;23(4):307-14

     9. PMID:   22053062
        Cox H et al. "Emollients, education and quality of life: the RCPCH care pathway for children with eczema."
        Arch Dis Child  2011 Nov;96 Suppl 2:i19-24

    10. PMID:   22132560
        Venter C et al. "First tastes."
        J Fam Health Care  2011 Sep-Oct;21(5):14-7

</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-database]<br>(Parameter 1)</td>
<td>list</td>
<td>NCBI database to search</td>
<td><table><tr><td>pubmed</td> <td><i>(pubmed)</i></td></tr><tr><td>nucleotide</td> <td><i>(nuccore)</i></td></tr><tr><td>est</td> <td><i>(nucest)</i></td></tr><tr><td>gss</td> <td><i>(nucgss)</i></td></tr><tr><td>protein</td> <td><i>(protein)</i></td></tr><tr><td>genome</td> <td><i>(genome)</i></td></tr><tr><td>structure</td> <td><i>(structure)</i></td></tr><tr><td>taxonomy</td> <td><i>(taxonomy)</i></td></tr><tr><td>snp</td> <td><i>(snp)</i></td></tr><tr><td>dbvar</td> <td><i>(dbvar)</i></td></tr><tr><td>gene</td> <td><i>(gene)</i></td></tr><tr><td>sra</td> <td><i>(sra)</i></td></tr><tr><td>biosystems</td> <td><i>(biosystems)</i></td></tr><tr><td>homologene</td> <td><i>(homologene)</i></td></tr><tr><td>probe</td> <td><i>(brobe)</i></td></tr><tr><td>bioproject</td> <td><i>(bioproject)</i></td></tr><tr><td>gap</td> <td><i>(gap)</i></td></tr><tr><td>cdd</td> <td><i>(cdd)</i></td></tr><tr><td>clone</td> <td><i>(clone)</i></td></tr><tr><td>unists</td> <td><i>(unists)</i></td></tr><tr><td>popset</td> <td><i>(popset)</i></td></tr><tr><td>geoprofiles</td> <td><i>(geoprofiles)</i></td></tr><tr><td>geodatasets</td> <td><i>(gds)</i></td></tr><tr><td>epigenomics</td> <td><i>(epigenomics)</i></td></tr><tr><td>bioassay</td> <td><i>(pcassay)</i></td></tr><tr><td>compound</td> <td><i>(pccompound)</i></td></tr><tr><td>substance</td> <td><i>(pcsubstance)</i></td></tr><tr><td>proteinclusters</td> <td><i>(proteinclusters)</i></td></tr><tr><td>omia</td> <td><i>(omia)</i></td></tr><tr><td>biosample</td> <td><i>(biosample)</i></td></tr></table></td>
<td>pubmed</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-query]<br>(Parameter 2)</td>
<td>string</td>
<td>Query to search</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-outfile]<br>(Parameter 3)</td>
<td>outfile</td>
<td>Output file name</td>
<td>Output file</td>
<td>$(database).$(query).gentrez</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   gentrez reads nucleotide sequences.
</p>

<h2>Output file format</h2>

<p>
   The output from gentrez is to a file or stdout.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platform for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Web Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>

<h2>Warnings</h2>

<p>
None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-0882 Japan

  Kazuharu Arakawa (gaou@sfc.keio.ac.jp)
  Institute for Advanced Biosciences, Keio University
  252-0882 Japan</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

