<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gfind_ori_ter </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emb
oss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gfind_ori_ter
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Predict the replication origin and terminus in bacterial genomes
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   Predicts the replicational origin and terminus in circular bacterial genomes,<br>
   by taking the vertices of cumulative skew graphs (GC, keto, or purine). See<br>
   Frank and Lobry (2000) for the basic idea behind thid algorithm (but also note<br>
   that this algorithm is different from that of Oriloc, which uses GC3 of genes). <br>
<br>
   Terminus of replication can be more accurate by using noise-reduction <br>
   filtering using Fourier spectrum of the GC skew. This low-pass filtering<br>
   can be applied using -filter option. See Arakawa et al. (2007) for details.<br>
<br>
 <br>
   G-language SOAP service is provided by the<br>
   Institute for Advanced Biosciences, Keio University.<br>
   The original web service is located at the following URL:<br>
<br>
   http://www.g-language.org/wiki/soap<br>
<br>
   WSDL(RPC/Encoded) file is located at:<br>
<br>
   http://soap.g-language.org/g-language.wsdl<br>
<br>
   Documentation on G-language Genome Analysis Environment methods are<br>
   provided at the Document Center<br>
<br>
   http://ws.g-language.org/gdoc/
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with gfind_ori_ter

% gfind_ori_ter NC_000913.gbk
Predict the replication origin and terminus in bacterial genomes
NC_000913
3922946	1550274

</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-window</td>
<td>integer</td>
<td>Number of windows to use for Fat Fourier Transform. Only active when -filter option is specified. Value must be the power of two</td>
<td>Any integer value</td>
<td>4096</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-purine</td>
<td>boolean</td>
<td>Use purine skew for calculation</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-keto</td>
<td>boolean</td>
<td>Use keto skew for calculation</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-filt</td>
<td>integer</td>
<td>Lowpass filter strength in percent. Typically 95 or 99 works best</td>
<td>Any integer value</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-accid</td>
<td>string</td>
<td>Include to use sequence accession ID as query</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   gfind_ori_ter reads nucleotide sequences.
</p>

<h2>Ouptut file format</h2>

<p>
   The output from gfind_ori_ter is to standard output.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
   Frank AC, Lobry JR (2000) "Oriloc: prediction of replication boundaries in unannotated 
      bacterial chromosomes", Bioinformatics, 16:566-567.

   Arakawa K, Saito R, Tomita M (2007) "Noise-reduction filtering for accurate detection 
      of replication termini in bacterial genomes", FEBS Letters, 581(2):253-258.

   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Wev Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>

<h2>Warnings</h2>

<p>
   None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

<tr>
<td><a href=" .html"> </a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   G-language SOAP service is provided by the.html">   G-language SOAP service is provided by the</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   Institute for Advanced Biosciences, Keio University..html">   Institute for Advanced Biosciences, Keio University.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   The original web service is located at the following URL:.html">   The original web service is located at the following URL:</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   http://www.g-language.org/wiki/soap.html">   http://www.g-language.org/wiki/soap</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   WSDL(RPC/Encoded) file is located at:.html">   WSDL(RPC/Encoded) file is located at:</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   http://soap.g-language.org/g-language.wsdl.html">   http://soap.g-language.org/g-language.wsdl</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   Documentation on G-language Genome Analysis Environment methods are.html">   Documentation on G-language Genome Analysis Environment methods are</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   provided at the Document Center.html">   provided at the Document Center</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   http://ws.g-language.org/gdoc/.html">   http://ws.g-language.org/gdoc/</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="Usage.html">Usage</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   Here is a sample session with gfind_ori_ter.html">   Here is a sample session with gfind_ori_ter</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="% gfind_ori_ter NC_000913.gbk.html">% gfind_ori_ter NC_000913.gbk</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="Predict the replication origin and terminus in bacterial genomes.html">Predict the replication origin and terminus in bacterial genomes</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="NC_000913.html">NC_000913</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="3922946	1550274.html">3922946	1550274</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="Command line arguments.html">Command line arguments</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="Standard (Mandatory) qualifiers:.html">Standard (Mandatory) qualifiers:</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="  [-sequence]          seqall     Nucleotide sequence(s) filename and optional.html">  [-sequence]          seqall     Nucleotide sequence(s) filename and optional</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="                                  format, or reference (input USA).html">                                  format, or reference (input USA)</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   Additional (Optional) qualifiers: (none).html">   Additional (Optional) qualifiers: (none)</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   Advanced (Unprompted) qualifiers:.html">   Advanced (Unprompted) qualifiers:</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   -window             integer    [4096] Number of windows to use for Fat.html">   -window             integer    [4096] Number of windows to use for Fat</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="                                  Fourier Transform. Only active when -filter.html">                                  Fourier Transform. Only active when -filter</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="                                  option is specified. Value must be the power.html">                                  option is specified. Value must be the power</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="                                  of two (Any integer value).html">                                  of two (Any integer value)</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   -purine             boolean    [0] Use purine skew for calculation.html">   -purine             boolean    [0] Use purine skew for calculation</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   -keto               boolean    [0] Use keto skew for calculation.html">   -keto               boolean    [0] Use keto skew for calculation</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   -filt               integer    [0] Lowpass filter strength in percent..html">   -filt               integer    [0] Lowpass filter strength in percent.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="                                  Typically 95 or 99 works best (Any integer.html">                                  Typically 95 or 99 works best (Any integer</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="                                  value).html">                                  value)</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   -accid              string     Include to use sequence accession ID as.html">   -accid              string     Include to use sequence accession ID as</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="                                  query (Any string).html">                                  query (Any string)</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   General qualifiers:.html">   General qualifiers:</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   -help               boolean    Report command line options and exit. More.html">   -help               boolean    Report command line options and exit. More</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="                                  information on associated and general.html">                                  information on associated and general</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="                                  qualifiers can be found with -help -verbose.html">                                  qualifiers can be found with -help -verbose</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="Input file format.html">Input file format</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   gfind_ori_ter reads nucleotide sequences..html">   gfind_ori_ter reads nucleotide sequences.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="Output file format.html">Output file format</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   The output from gfind_ori_ter is to standard output..html">   The output from gfind_ori_ter is to standard output.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="Data files.html">Data files</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   None..html">   None.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="Notes.html">Notes</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   None..html">   None.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="References.html">References</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   Frank AC, Lobry JR (2000) "Oriloc: prediction of replication boundaries in unannotated .html">   Frank AC, Lobry JR (2000) "Oriloc: prediction of replication boundaries in unannotated </a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="      bacterial chromosomes", Bioinformatics, 16:566-567..html">      bacterial chromosomes", Bioinformatics, 16:566-567.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   Arakawa K, Saito R, Tomita M (2007) "Noise-reduction filtering for accurate detection .html">   Arakawa K, Saito R, Tomita M (2007) "Noise-reduction filtering for accurate detection </a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="      of replication termini in bacterial genomes", FEBS Letters, 581(2):253-258..html">      of replication termini in bacterial genomes", FEBS Letters, 581(2):253-258.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and.html">   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench.html">      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306..html">      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for.html">   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="      large-scale analysis of high-throughput omics data, J. Pest Sci.,.html">      large-scale analysis of high-throughput omics data, J. Pest Sci.,</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="      31, 7..html">      31, 7.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome.html">   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="      Analysis Environment with REST and SOAP Wev Service Interfaces,.html">      Analysis Environment with REST and SOAP Wev Service Interfaces,</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="      Nucleic Acids Res., 38, W700-W705..html">      Nucleic Acids Res., 38, W700-W705.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="Warnings.html">Warnings</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   None..html">   None.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="Diagnostic Error Messages.html">Diagnostic Error Messages</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   None..html">   None.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="Exit status.html">Exit status</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   It always exits with a status of 0..html">   It always exits with a status of 0.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="Known bugs.html">Known bugs</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="   None..html">   None.</a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href=".html"></a></td>
<td>appdescription</td>
</tr>

<tr>
<td><a href="See also.html">See also</a></td>
<td>appdescription</td>
</tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa(gaou@g-language.org) 
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

