<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gfind_ori_ter </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emb
oss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gfind_ori_ter
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Predict the replication origin and terminus in bacterial genomes
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   Predicts the replicational origin and terminus in circular bacterial genomes,<br>
   by taking the vertices of cumulative skew graphs (GC, keto, or purine). See<br>
   Reference 1 for the basic idea behind thid algorithm (but also note that this<br>
   algorithm is different from that of Oriloc, which uses GC3 of genes). <br>
<br>
   Terminus of replication can be more accurate by using noise-reduction <br>
   filtering using Fourier spectrum of the GC skew. This low-pass filtering<br>
   can be applied using -filter option. See Reference 2 for details.<br>
<br>
 <br>
   G-language SOAP service is provided by the<br>
   Institute for Advanced Biosciences, Keio University.<br>
   The original web service is located at the following URL:<br>
<br>
   http://www.g-language.org/wiki/soap<br>
<br>
   WSDL(RPC/Encoded) file is located at:<br>
<br>
   http://soap.g-language.org/g-language.wsdl<br>
<br>
   Documentation on G-language Genome Analysis Environment methods are<br>
   provided at the Document Center<br>
<br>
   http://ws.g-language.org/gdoc/
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with gfind_ori_ter

% gfind_ori_ter  NC_000913.gbk
Predict the replication origin and terminus in bacterial genomes
Output filename [*.gfind_ori_ter]:


   Creates *.gfind_ori_ter
   Output example is prepared in the data directory

</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-outfile</td>
<td>outfile</td>
<td>Output file name</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.gfind_ori_ter</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-window</td>
<td>integer</td>
<td>Number of windows to use for Fat Fourier Transform. Only active when -filter option is specified. Value must be the power of two</td>
<td>Any integer value</td>
<td>4096</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-purine</td>
<td>boolean</td>
<td>Use purine skew for calculation</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-keto</td>
<td>boolean</td>
<td>Use keto skew for calculation</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-filt</td>
<td>integer</td>
<td>Lowpass filter strength in percent. Typically 95 or 99 works best</td>
<td>Any integer value</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-accid</td>
<td>string</td>
<td>Include to use sequence accession ID as query</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-show</td>
<td>toggle</td>
<td>Include to dump result to stdout</td>
<td>Toggle value Yes/No</td>
<td>No</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   gfind_ori_ter reads nucleotide sequences.
</p>

<h2>Ouptut file format</h2>

<p>
   Creates *.gfind_ori_ter
   Output example is prepared in the data directory

Command line arguments

Standard (Mandatory) qualifiers (* if not always prompted):
  [-sequence]          seqall     Nucleotide sequence(s) filename and optional
                                  format, or reference (input USA)
*  -outfile            outfile    [*.gfind_ori_ter] Output file name

   Additional (Optional) qualifiers: (none)
   Advanced (Unprompted) qualifiers:
   -window             integer    [4096] Number of windows to use for Fat
                                  Fourier Transform. Only active when -filter
                                  option is specified. Value must be the power
                                  of two (Any integer value)
   -purine             boolean    [0] Use purine skew for calculation
   -keto               boolean    [0] Use keto skew for calculation
   -filt               integer    [0] Lowpass filter strength in percent.
                                  Typically 95 or 99 works best (Any integer
                                  value)
   -accid              string     Include to use sequence accession ID as
                                  query (Any string)
   -show               toggle     [N] Include to dump result to stdout

   General qualifiers:
   -help               boolean    Report command line options and exit. More
                                  information on associated and general
                                  qualifiers can be found with -help -verbose


Input file format

   gfind_ori_ter reads nucleotide sequences.

Output file format

   The output from gfind_ori_ter is to a file or stdout.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
   Frank AC, Lobry JR (2000) "Oriloc: prediction of replication boundaries in unannotated 
      bacterial chromosomes", Bioinformatics, 16:566-567.

   Arakawa K, Saito R, Tomita M (2007) "Noise-reduction filtering for accurate detection 
      of replication termini in bacterial genomes", FEBS Letters, 581(2):253-258.

   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Wev Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>

<h2>Warnings</h2>

<p>
   None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

<tr>
<td><a href="grep_ori_ter.html">grep_ori_ter</a></td>
<td>Get the positions of replication origin and terminus</td>
</tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa(gaou@g-language.org) 
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

