<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: ggcsi </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emb
oss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
ggcsi
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   GC Skew Index: an index for strand-specefic mutational bias
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   This program calculates the GC Skew Index (GCSI) of the given circular<br>
   bacterial genome. GCSI quantifies the degree of GC Skew. In other words,<br>
   this index represents the degree of strand-specific mutational bias in <br>
   bacterial genomes, caused by replicational selection. <br>
<br>
   GCSI is calculated by the following formula:<br>
   GCSI = sqrt((SA/6000) * (dist/600))<br>
   where SA is the spectral amplitude of Fourier power spectrum at 1Hz,<br>
   and dist is the normalized Euclidean distance between the vertices of <br>
   cumulative GC skew.<br>
<br>
   GCSI ranges from 0 (no observable skew) to 1 (strong skew), and Archaeal<br>
   genomes that have multiple replication origins and therefore have no<br>
   observable skew mostly have GCSI below 0.05. Escherichia coli genome has<br>
   values around 0.10.<br>
<br>
   Version 1 of GCSI required fixed number of windows (4096), but the new GCSI<br>
   version 2 (also known as generalized GCSI: gGCSI) is invariant of the number<br>
   of windows. GCSI version 1 is calculated as an arithmetic mean (as opposed to <br>
   the geometric mean of gGCSI) of SR (spectral ratio, the signal-to-noise ratio<br>
   of 1Hz power spectrum) and dist. <br>
<br>
   G-language SOAP service is provided by the<br>
   Institute for Advanced Biosciences, Keio University.<br>
   The original web service is located at the following URL:<br>
<br>
   http://www.g-language.org/wiki/soap<br>
<br>
   WSDL(RPC/Encoded) file is located at:<br>
<br>
   http://soap.g-language.org/g-language.wsdl<br>
<br>
   Documentation on G-language Genome Analysis Environment methods are<br>
   provided at the Document Center<br>
<br>
   http://ws.g-language.org/gdoc/
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with ggcsi

   The database definitions for following commands are available at
   http://soap.g-language.org/kbws/embossrc

% ggcsi -show refseqn:NC_000913
GC Skew Index: an index for strand-specefic mutational bias
Sequence: NC_000913 GCSI: 0.0966615833014818 SA: 487.218569030757 DIST: 69.037726

</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-outfile</td>
<td>outfile</td>
<td>Output file name</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.ggcsi</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-gcsi</td>
<td>boolean</td>
<td>Include to use GCSI version 1</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-window</td>
<td>integer</td>
<td>Number of windows. Must be a power of 2</td>
<td>Any integer value</td>
<td>4096</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-purine</td>
<td>boolean</td>
<td>Use purine skew for calculation</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-keto</td>
<td>boolean</td>
<td>Use keto skew for calculation</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-at</td>
<td>boolean</td>
<td>Use AT skew for calculation</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-pval</td>
<td>boolean</td>
<td>Calculate p-value when GCSI version 2 is selected</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-accid</td>
<td>string</td>
<td>Include to use sequence accession ID as query</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-show</td>
<td>toggle</td>
<td>Include to dump result to stdout</td>
<td>Toggle value Yes/No</td>
<td>No</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   ggcsi reads nucleotide sequences.
</p>

<h2>Ouptut file format</h2>

<p>
   Additional (Optional) qualifiers: (none)
   Advanced (Unprompted) qualifiers:
   -gcsi               boolean    [N] Include to use GCSI version 1
   -window             integer    [4096] Number of windows. Must be a power of
                                  2 (Any integer value)
   -purine             boolean    [0] Use purine skew for calculation
   -keto               boolean    [0] Use keto skew for calculation
   -at                 boolean    [0] Use AT skew for calculation
   -pval               boolean    [0] Calculate p-value when GCSI version 2 is
                                  selected
   -accid              string     Include to use sequence accession ID as
                                  query (Any string)
   -show               toggle     [N] Include to dump result to stdout

   General qualifiers:
   -help               boolean    Report command line options and exit. More
                                  information on associated and general
                                  qualifiers can be found with -help -verbose

Input file format

   ggcsi reads nucleotide sequences.

Output file format

   The output from ggcsi is to a file or stdout.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
   Arakawa K, Tomita M (2007) "The GC skew index: a measure of genomic compositional 
      asymmetry and the degree of replicational selection", Evolutionary Bioinformatics, 3:145-154.

   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Wev Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>

<h2>Warnings</h2>

<p>

</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

<tr>
<td><a href="gb1.html">gb1</a></td>
<td>Calculate strand bias of bacterial genome using B1 index</td>
</tr>

<tr>
<td><a href="gb2.html">gb2</a></td>
<td>Calculate strand bias of bacterial genome using B2 index</td>
</tr>

<tr>
<td><a href="gdelta_gcskew.html">gdelta_gcskew</a></td>
<td>Calculate strand bias of bacterial genome using delta GC skew index</td>
</tr>

<tr>
<td><a href="glda_bias.html">glda_bias</a></td>
<td>Calculate strand bias of bacterial genome using linear discriminant analysis (LDA)</td>
</tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa(gaou@g-language.org) 
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

