<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: ggenomicskew </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="http://soap.g-language.org/gembassy/emboss_explorer/manual/emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
ggenomicskew
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Calculates the GC skew in different regions of the given genome
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   ggenomicskew calculates and plots the GC skew for the whole genome, coding<br />
   regions, intergenic regions, and the third codon. This program is useful in<br />
   visualizing various base composition bias within the genome. AT skew can be<br />
   calculated instead of GC skew by toggling the -at qualifier.<br />
    <br />
   G-language SOAP service is provided by the<br />
   Institute for Advanced Biosciences, Keio University.<br />
   The original web service is located at the following URL:<br />
<br />
   http://www.g-language.org/wiki/soap<br />
<br />
   WSDL(RPC/Encoded) file is located at:<br />
<br />
   http://soap.g-language.org/g-language.wsdl<br />
<br />
   Documentation on G-language Genome Analysis Environment methods are<br />
   provided at the Document Center<br />
<br />
   http://ws.g-language.org/gdoc/<br />
<br />

</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

Here is a sample session with ggenomicskew

% ggenomicskew refseqn:NC_000913
Calculates the GC skew in different regions of the given genome
Program compseq output file (optional) [nc_000913.ggenomicskew]: 

   Go to the input files for this example
   Go to the output files for this example

   Example 2

% ggenomicskew refseqn:NC_000913 -plot -graph png
Calculates the GC skew in different regions of the given genome
Created ggenomicskew.1.png

   Go to the input files for this example
   Go to the output files for this example



</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-graph</td>
<td>xygraph</td>
<td>Graph type</td>
<td>EMBOSS has a list of known devices, including ps, hpgl, hp7470, hp7580, meta, cps, x11, tek, tekt, none, data, xterm, png, gif, svg</td>
<td><i>EMBOSS_GRAPHICS</i> value, or x11</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-outfile</td>
<td>outfile</td>
<td>Program compseq output file (optional)</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.ggenomicskew</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-divide</td>
<td>integer</td>
<td>Window to divide into</td>
<td>Any integer value</td>
<td>250</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-at</td>
<td>boolean</td>
<td>Input 1 when observing AT skew instead of GC skew</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-[no]accid</td>
<td>boolean</td>
<td>Include to use sequence accession ID as query</td>
<td>Boolean value Yes/No</td>
<td>Yes</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-plot</td>
<td>toggle</td>
<td>Include to plot result</td>
<td>Toggle value Yes/No</td>
<td>No</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   The database definitions for following commands are available at<br />
   http://soap.g-language.org/kbws/embossrc<br />
<br />
   ggenomicskew reads one or more nucleotide sequences.<br />
<br />

</p>

<h2>Output file format</h2>

<p>
   The output from ggenomicskew is to a plain text file or the EMBOSS graphics device.<br />
<br />
   File: nc_000913.ggenomicskew<br />
<br />
Sequence: NC_000913<br />
location,GC skew,coding,intergenic,third codon<br />
0,-0.036259,-0.040085,-0.034707,-0.141888,<br />
1,-0.031167,-0.035657,0.047953,-0.175758,<br />
2,-0.028670,-0.031139,-0.049143,-0.018466,<br />
3,-0.016647,-0.004656,-0.102616,0.086181,<br />
4,-0.041985,-0.029846,-0.088670,0.015291,<br />
5,-0.097093,-0.103813,-0.067275,-0.247401,<br />
6,-0.028028,-0.016363,-0.048806,-0.047332,<br />
7,-0.055805,-0.059329,-0.020071,-0.123271,<br />
<br />
   [Part of this file has been deleted for brevity]<br />
<br />
241,0.000772,-0.012151,-0.063786,0.069585,<br />
242,-0.025787,-0.000384,-0.049143,0.029431,<br />
243,0.010516,0.008217,-0.030600,0.128657,<br />
244,-0.037115,-0.015134,0.017500,0.035398,<br />
245,-0.000317,0.006021,-0.047170,0.091549,<br />
246,-0.025417,-0.015190,-0.116608,0.044619,<br />
247,-0.038404,-0.035676,-0.135714,0.015375,<br />
248,-0.026246,-0.024240,-0.037190,-0.130118,<br />
249,-0.053371,-0.057225,-0.022472,-0.082167,<br />
250,-0.026316,0.166667,-0.151515,0.000000,<br />
<br />
<br />

</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>References</h2>

<pre>
   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platform for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Web Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.

</pre>

<h2>Warnings</h2>

<p>
None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

<tr>
<td><a href="ggcskew.html">ggcskew</a></td>
<td>Calculates the GC skew of the input sequence</td>
</tr><tr>
<td><a href="ggcwin.html">ggcwin</a></td>
<td>Calculates the GC content along the given genome</td>
</tr><tr>
<td><a href="ggeneskew.html">ggeneskew</a></td>
<td>Calculate the gene strand bias of the given genome</td>
</tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-0882 Japan

  Kazuharu Arakawa (gaou@sfc.keio.ac.jp)
  Institute for Advanced Biosciences, Keio University
  252-0882 Japan</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

