<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gkmertable </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="http://soap.g-language.org/gembassy/emboss_explorer/manual/emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gkmertable
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Create an image showing all k-mer abundance within a sequence
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   gkmertable creates an image showing the abundance of all k-mers<br />
   (oligonucleotides of length k) in a given sequence. For example, for<br />
   tetramers (k=4), resulting image is composed of 4^4 = 256 boxes, each<br />
   representing an oligomer. Oligomer name and abundance is written within<br />
   these boxes, and abundance is also visualized with the box color, from<br />
   white (none) to black (highly frequent).<br />
   This k-mer table is alternatively known as the FCGR (frequency matrices<br />
   extracted from Chaos Game Representation).<br />
   Position of the oligomers can be recursively located as follows:<br />
   For each letter in an oligomer, a box is subdivided into four quadrants, <br />
   where A is upper left, T is lower right, G is upper right, and C is lower<br />
   left.<br />
   Therefore, oligomer ATGC is in the <br />
   A = upper left quadrant<br />
   T = lower right within the above quadrant<br />
   G = upper right within the above quadrant<br />
   C = lower left within the above quadrant<br />
   More detailed documentation is available at <br />
   http://www.g-language.org/wiki/cgr<br />
    <br />
   G-language SOAP service is provided by the<br />
   Institute for Advanced Biosciences, Keio University.<br />
   The original web service is located at the following URL:<br />
<br />
   http://www.g-language.org/wiki/soap<br />
<br />
   WSDL(RPC/Encoded) file is located at:<br />
<br />
   http://soap.g-language.org/g-language.wsdl<br />
<br />
   Documentation on G-language Genome Analysis Environment methods are<br />
   provided at the Document Center<br />
<br />
   http://ws.g-language.org/gdoc/<br />
<br />

</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

Here is a sample session with gkmertable

% gkmertable refseqn:NC_000913
Create an image showing all k-mer abundance within a sequence
Output file for non interactive displays [gkmertable]: Created gkmertable.1.png


   Go to the input files for this example
   Go to the output files for this example



</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-goutfile]<br>(Parameter 2)</td>
<td>string</td>
<td>Output file for non interactive displays</td>
<td>Any string</td>
<td>gkmertable</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-format</td>
<td>string</td>
<td>Output file format. Dependent on 'convert' command</td>
<td>Any string</td>
<td>png</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-k</td>
<td>integer</td>
<td>Length of oligomer</td>
<td>Any integer value</td>
<td>6</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   The database definitions for following commands are available at<br />
   http://soap.g-language.org/kbws/embossrc<br />
<br />
   gkmertable reads one or more nucleotide sequences.<br />
<br />

</p>

<h2>Output file format</h2>

<p>
   The output from gkmertable is to an image file.<br />
<br />
<br />
<br />

</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>References</h2>

<pre>
   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platform for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Web Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.

</pre>

<h2>Warnings</h2>

<p>
None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

<tr>
<td><a href="gkmer_table.html">gkmer_table</a></td>
<td>Create an image showing all k-mer abundance within a</td>
</tr><tr>
<td><a href="gnucleotide_periodicity.html">gnucleotide_periodicity</a></td>
<td>Checks the periodicity of certain oligonucleotides</td>
</tr><tr>
<td><a href="goligomer_counter.html">goligomer_counter</a></td>
<td>Counts the number of given oligomers in a sequence</td>
</tr><tr>
<td><a href="goligomer_search.html">goligomer_search</a></td>
<td>Searches oligomers in given sequence</td>
</tr><tr>
<td><a href="gsignature.html">gsignature</a></td>
<td>Calculate oligonucleotide usage (genomic signature)</td>
</tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-0882 Japan

  Kazuharu Arakawa (gaou@sfc.keio.ac.jp)
  Institute for Advanced Biosciences, Keio University
  252-0882 Japan</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

