<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gkmer_table </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emb
oss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gkmer_table
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Create an image showing all k-mer abundance within a sequence
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   This program creates an image showing the abundance of all k-mers<br>
   (oligonucleotides of length k) in a given sequence. For example, for<br>
   tetramers (k=4), resulting image is composed of 4^4 = 256 boxes, each<br>
   representing an oligomer. Oligomer name and abundance is written within<br>
   these boxes, and abundance is also visualized with the box color, from white<br>
   (none) to black (highly frequent).<br>
<br>
   This k-mer table is alternatively known as the FCGR (frequency matrices<br>
   extracted from Chaos Game Representation).<br>
<br>
   Position of the oligomers can be recursively located as follows:<br>
   For each letter in an oligomer, a box is subdivided into four quadrants, <br>
   where A is upper left, T is lower right, G is upper right, and C is lower<br>
   left.<br>
   Therefore, oligomer ATGC is in the <br>
     A = upper left quadrant<br>
     T = lower right within the above quadrant<br>
     G = upper right within the above quadrant<br>
     C = lower left within the above quadrant<br>
<br>
<br>
   More detailed documentation is available at <br>
   http://www.g-language.org/wiki/cgr<br>
 <br>
  <br>
   G-language SOAP service is provided by the<br>
   Institute for Advanced Biosciences, Keio University.<br>
   The original web service is located at the following URL:<br>
<br>
   http://www.g-language.org/wiki/soap<br>
<br>
   WSDL(RPC/Encoded) file is located at:<br>
<br>
   http://soap.g-language.org/g-language.wsdl<br>
<br>
   Documentation on G-language Genome Analysis Environment methods are<br>
   provided at the Document Center<br>
<br>
   http://ws.g-language.org/gdoc/
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with gkmer_table

% gkmer_table  NC_000913.gbk
Create an image showing all k-mer abundance within a sequence
Output filename [gkmer_table.[accession].png]:

   Creates gkmer_table.[accession].png
   Output example is prepared in the data directory

</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-filename</td>
<td>string</td>
<td>Output filename</td>
<td>Any string</td>
<td>gkmer_table.[accession].png</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-k</td>
<td>integer</td>
<td>Length of oligomer</td>
<td>Any integer value</td>
<td>6</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-show</td>
<td>boolean</td>
<td>Include to show png with X11 window</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-accid</td>
<td>string</td>
<td>Include to use sequence accession ID as query</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   gkmer_table reads nucleotide sequences.
</p>

<h2>Ouptut file format</h2>

<p>
   Creates gkmer_table.[accession].png
   Output example is prepared in the data directory

Command line arguments

Standard (Mandatory) qualifiers:
  [-sequence]          seqall     Nucleotide sequence(s) filename and optional
                                  format, or reference (input USA)
   -filename           string     [gkmer_table.[accession].png] Output
                                  filename (Any string)

   Additional (Optional) qualifiers: (none)
   Advanced (Unprompted) qualifiers:
   -k                  integer    [6] Length of oligomer (Any integer value)
   -show               boolean    [N] Include to show png with X11 window
   -accid              string     Include to use sequence accession ID as
                                  query (Any string)

   General qualifiers:
   -help               boolean    Report command line options and exit. More
                                  information on associated and general
                                  qualifiers can be found with -help -verbose


Input file format

   gkmer_table reads nucleotide sequences.

Output file format

   The output from gkmer_table is to PNG format.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
   Arakawa K, Oshita K, Tomita M (2009) "A web server for interactive and
      zoomable Chaos Game Representation images", Source Code for Biol Med.,
      4:6.

   Almeida JS, Carrico JA, Maretzek A, Noble PA, Fletcher M (2001) 
      "Analysis of genomic sequences by Chaos Game Representation." 
      Bioinformatics. 17(5):429-437.

   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Wev Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>

<h2>Warnings</h2>

<p>
   None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

<tr>
<td><a href="gkmer_table.html">gkmer_table</a></td>
<td>Create an image showing all k-mer abundance within a sequence</td>
</tr>

<tr>
<td><a href="gnucleotide_periodicity.html">gnucleotide_periodicity</a></td>
<td>Checks the periodicity of certain oligonucleotides</td>
</tr>

<tr>
<td><a href="goligomer_counter.html">goligomer_counter</a></td>
<td>Counts the number of given oligomers in a sequence</td>
</tr>

<tr>
<td><a href="goligomer_search.html">goligomer_search</a></td>
<td>Searches oligomers in given sequence</td>
</tr>

<tr>
<td><a href="gsignature.html">gsignature</a></td>
<td>Calculate oligonucleotide usage (genomic signature)</td>
</tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa(gaou@g-language.org) 
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

