<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gkmer_table </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gkmer_table
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
Create an image showing all k-mer abundance within a sequence
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   This program creates an image showing the abundance of all k-mers (oligonucleotides
   of length k) in a given sequence. For example, for tetramers (k=4), resulting image
   is composed of 4^4 = 256 boxes, each representing an oligomer. Oligomer name and 
   abundance is written within these boxes, and abundance is also visualized with 
   the box color, from white (none) to black (highly frequent).

   This k-mer table is alternatively known as the FCGR (frequency matrices extracted
   from Chaos Game Representation).

   Position of the oligomers can be recursively located as follows:
   For each letter in an oligomer, a box is subdivided into four quadrants, 
   where A is upper left, T is lower right, G is upper right, and C is lower left.
   Therefore, oligomer ATGC is in the 
   A = upper left quadrant
   T = lower right within the above quadrant
   G = upper right within the above quadrant
   C = lower left within the above quadrant

   With Google Maps representation using -gmap=>1 option, oligomers can be searched
   incrementally from the top search box.

   More detailed documentation is available at 
   http://www.g-language.org/wiki/cgr
 
  
</p>

<p>
G-language SOAP service is provided by the Institute for Advanced Biosciences, Keio University.<br />

The original web service is located at the following URL:<br />
http://www.g-language.org/wiki/soap
</p>

<p>
WSDL(RPC/Encoded) file is located at:<br />
http://soap.g-language.org/g-language.wsdl
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with gkmer_table

% gkmer_table NC_000913.gbk
NC_000913
Retrieval successful


</pre></td></tr></table>



<h2>Command line argumants</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-k</td>
<td>integer</td>
<td>Length of oligomer</td>
<td>Any integer value</td>
<td>6</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-accid</td>
<td>boolean</td>
<td>Include to use sequence accession ID as query</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
gkmer_table reads nucleotide sequences.
</p>

<h2>Ouptput file format</h2>

<p>
The output from gkmer_table is to PNG format.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
    1. Arakawa K, Oshita K, Tomita M (2009) "A web server for interactive and
       zoomable Chaos Game Representation images", Source Code for Biol Med., 4:6.
    2. Almeida JS, Carrico JA, Maretzek A, Noble PA, Fletcher M (2001) 
       "Analysis of genomic sequences by Chaos Game Representation." 
       Bioinformatics. 17(5):429-437.
  
  Author: 
    Kazuharu Arakawa (gaou@sfc.keio.ac.jp)
    
  History:
    20090716-01 initial posting

   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Wev Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>
<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa (gaou@sfc.keio.ac.jp)
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

    None.

