<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: grep_ori_ter </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emb
oss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
grep_ori_ter
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Get the positions of replication origin and terminus
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   This method returns the positions of replication origin and terminus<br>
   in bacterial genomes by several means. <br>
<br>
   1. Use of databases<br>
   By default, rep_ori_ter() tries to retrieve the position of replication<br>
   origin in DoriC (Reference 1) database, and the position of replication<br>
   terminus from the supplemental data provided in Reference 2.<br>
<br>
   If the position of origin cannot be found in the database, but "rep_origin" <br>
   feature is available, center position within this feature is used for origin. <br>
<br>
   2. Oriloc<br>
   Using -oriloc=>1 option, you can predict the replication origin and <br>
   terminus using the popular Oriloc program developed by Lobry et al. <br>
   available as part of the SeqinR package (Reference 3).<br>
<br>
   Installation of SeqinR package is required to use this feature.<br>
   Type: install.packages('seqinr', dep=T)<br>
   in R statistics package to install SeqinR.<br>
<br>
   3. Setting custom origin and terminus<br>
   Use -ori=>$position1, -ter=>$position2 to set your own positions.<br>
<br>
   4. Use GC skew shift-point<br>
   If the positions of origin or terminus cannot be found in the databases,<br>
   rep_ori_ter() automatically calls find_ori_ter() method to predict the <br>
   positions using GC skew shift-points at one-base-pair resolution.<br>
<br>
   After calling this function, positions of origin and terminus are stored <br>
   as follows:<br>
   $genome->{FEATURE0}->{origin}<br>
   $genome->{FEATURE0}->{terminus}<br>
<br>
   You can always use -clear=>1 option to disable the use of above cache.<br>
<br>
 <br>
   G-language SOAP service is provided by the<br>
   Institute for Advanced Biosciences, Keio University.<br>
   The original web service is located at the following URL:<br>
<br>
   http://www.g-language.org/wiki/soap<br>
<br>
   WSDL(RPC/Encoded) file is located at:<br>
<br>
   http://soap.g-language.org/g-language.wsdl<br>
<br>
   Documentation on G-language Genome Analysis Environment methods are<br>
   provided at the Document Center<br>
<br>
   http://ws.g-language.org/gdoc/
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with grep_ori_ter

% grep_ori_ter  NC_000913.gbk
Get the positions of replication origin and terminus
Output filename [*.grep_ori_ter]:


   Creates *.grep_ori_ter
   Output example is prepared in the data directory

</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-outfile</td>
<td>outfile</td>
<td>Output file name</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.grep_ori_ter</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-oriloc</td>
<td>boolean</td>
<td>Set to 1 to use Oriloc for prediction</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-gcskew</td>
<td>boolean</td>
<td>Set to 1 to use GC skew shift-point for prediction</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-difthreshold</td>
<td>integer</td>
<td>Distance between the GC skew shift point and predicted dif site expressed as the precentage of genome size, used as a threshold to retrieve dif sequence from the database</td>
<td>Any integer value</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-dbonly</td>
<td>boolean</td>
<td>Set to 1 to only use values available in databases and to suppress prediction</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-accid</td>
<td>string</td>
<td>Include to use sequence accession ID as query</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-show</td>
<td>toggle</td>
<td>Include to dump result to stdout</td>
<td>Toggle value Yes/No</td>
<td>No</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   grep_ori_ter reads nucleotide sequences.
</p>

<h2>Ouptut file format</h2>

<p>
   Creates *.grep_ori_ter
   Output example is prepared in the data directory

Command line arguments

Standard (Mandatory) qualifiers (* if not always prompted):
  [-sequence]          seqall     Nucleotide sequence(s) filename and optional
                                  format, or reference (input USA)
*  -outfile            outfile    [*.grep_ori_ter] Output file name

   Additional (Optional) qualifiers: (none)
   Advanced (Unprompted) qualifiers:
   -oriloc             boolean    [0] Set to 1 to use Oriloc for prediction
   -gcskew             boolean    [0] Set to 1 to use GC skew shift-point for
                                  prediction
   -difthreshold       integer    [0] Distance between the GC skew shift point
                                  and predicted dif site expressed as the
                                  precentage of genome size, used as a
                                  threshold to retrieve dif sequence from the
                                  database (Any integer value)
   -dbonly             boolean    [0] Set to 1 to only use values available in
                                  databases and to suppress prediction
   -accid              string     Include to use sequence accession ID as
                                  query (Any string)
   -show               toggle     [N] Include to dump result to stdout

   General qualifiers:
   -help               boolean    Report command line options and exit. More
                                  information on associated and general
                                  qualifiers can be found with -help -verbose


Input file format

   grep_ori_ter reads nucleotide sequences.

Output file format

   The output from grep_ori_ter is to a file or stdout.
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
   Gao F and Zhang CT (2007) "DoriC: a database of oriC regions in bacterial
      genomes", Bioinformatics, 23(14):1866-1867

   Kono N et al. (2011) "Comprehensive prediction of chromosome dimer resolution 
      sites in bacterial genomes", BMC Genomics, 12(1):19

   Frank AC and Lobry JR (2000) "Oriloc: prediction of replication boundaries 
      in unannotated bacterial chromosomes", Bioinformatics, 16(6):560-561

   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Wev Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>

<h2>Warnings</h2>

<p>
   None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

<tr>
<td><a href="gfind_ori_ter.html">gfind_ori_ter</a></td>
<td>Predict the replication origin and terminus in bacterial genomes</td>
</tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa(gaou@g-language.org) 
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

