<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gshuffleseq </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emb
oss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gshuffleseq
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Create randomized sequence with conserved k-mer composition
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   Shuffle and randomize the given sequence, conserving the nucleotide/peptide<br>
   k-mer content of the original sequence. <br>
<br>
   For k=1, i.e. shuffling sequencing preserving single nucleotide composition, <br>
   Fisher-Yates Algorithm is employed. <br>
<br>
   For k>1, shuffling preserves all k-mers (all k where k=1~k). For example,<br>
   k=3 preserves all triplet, doublet, and single nucleotide composition. <br>
   Algorithm for k-mer preserved shuffling is non-trivial, which is solved<br>
   by graph theoretical approach with Eulerian random walks in the graph of <br>
   k-1-mers. See references 3-5 for details of this algorithm. <br>
   <br>
   G-language SOAP service is provided by the<br>
   Institute for Advanced Biosciences, Keio University.<br>
   The original web service is located at the following URL:<br>
<br>
   http://www.g-language.org/wiki/soap<br>
<br>
   WSDL(RPC/Encoded) file is located at:<br>
<br>
   http://soap.g-language.org/g-language.wsdl<br>
<br>
   Documentation on G-language Genome Analysis Environment methods are<br>
   provided at the Document Center<br>
<br>
   http://ws.g-language.org/gdoc/
</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

   Here is a sample session with gshuffleseq

% gshuffleseq refseqn:NC_000913
Create randomized sequence with conserved k-mer composition
Output filename [gshuffleseq.nc_000913.csv]:
Died: Unknown qualifier -accid

</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-k</td>
<td>integer</td>
<td>k-mer to preserve composition</td>
<td>Any integer value</td>
<td>1</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-outseq</td>
<td>seqout</td>
<td>Sequence filename and optional format (output USA)</td>
<td>Writeable sequence</td>
<td><i>&lt;*&gt;</i>.<i>format</i></td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   gshuffleseq reads nucleotide sequences.
</p>

<h2>Ouptut file format</h2>

<p>
   The output from gshuffleseq is to .
</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>Refrences</h2>

<pre>
   Fisher R.A. and Yates F. (1938) "Example 12", Statistical Tables, London

   Durstenfeld R. (1964) "Algorithm 235: Random permutation", CACM 7(7):420

   Jiang M., Anderson J., Gillespie J., and Mayne M. (2008) "uShuffle: 
      a useful tool for shuffling biological sequences while preserving the k-let
      counts", BMC Bioinformatics 9:192

   Kandel D., Matias Y., Unver R., and Winker P. (1996) "Shuffling biological 
      sequences", Discrete Applied Mathematics 71(1-3):171-185  

   Propp J.G. and Wilson D.B. (1998) "How to get a perfectly random sample 
      from a generic Markov chain and generate a random spanning tree of a 
      directed graph", Journal of Algorithms 27(2):170-217

   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platfor for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Wev Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.
</pre>

<h2>Warnings</h2>

<p>

</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-8520 Japan

   Kazuharu Arakawa(gaou@g-language.org) 
</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

