<!--START OF HEADER - DON'T ALTER -->

<HTML>
<HEAD>
  <TITLE> EMBOSS: gviewcds </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">



<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="http://soap.g-language.org/gembassy/emboss_explorer/manual/emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
gviewcds
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>


<!--END OF HEADER-->






<H2> Function </H2>
   Displays a graph of nucleotide contents around start and stop codons
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>Description</H2>
<p>
   gviewcds creates a graph showing the average A,T,G,C contents<br />
   around start/stop codons. This is useful to view consensus around<br />
   start/stop codons and to find characteristic pattern in CDS.<br />
    <br />
   G-language SOAP service is provided by the<br />
   Institute for Advanced Biosciences, Keio University.<br />
   The original web service is located at the following URL:<br />
<br />
   http://www.g-language.org/wiki/soap<br />
<br />
   WSDL(RPC/Encoded) file is located at:<br />
<br />
   http://soap.g-language.org/g-language.wsdl<br />
<br />
   Documentation on G-language Genome Analysis Environment methods are<br />
   provided at the Document Center<br />
<br />
   http://ws.g-language.org/gdoc/<br />
<br />

</p>

<H2>Usage</H2>

<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

Here is a sample session with gviewcds

% gviewcds refseqn:NC_000913
Displays a graph of nucleotide contents around start and stop codons
Program compseq output file (optional) [nc_000913.gviewcds]: 

   Go to the input files for this example
   Go to the output files for this example

   Example 2

% gviewcds refseqn:NC_000913 -plot -graph png
Displays a graph of nucleotide contents around start and stop codons
Created gviewcds.1.png

   Go to the input files for this example
   Go to the output files for this example



</pre></td></tr></table>



<h2>Command line arguments</h2>

<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequence]<br>(Parameter 1)</td>
<td>seqall</td>
<td>Nucleotide sequence(s) filename and optional format, or reference (input USA)</td>
<td>Readable sequence(s)</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-graph</td>
<td>xygraph</td>
<td>Graph type</td>
<td>EMBOSS has a list of known devices, including ps, hpgl, hp7470, hp7580, meta, cps, x11, tek, tekt, none, data, xterm, png, gif, svg</td>
<td><i>EMBOSS_GRAPHICS</i> value, or x11</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-outfile</td>
<td>outfile</td>
<td>Program compseq output file (optional)</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.gviewcds</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-length</td>
<td>integer</td>
<td>Length in bases to show around start/stop codons</td>
<td>Any integer value</td>
<td>100</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-gap</td>
<td>integer</td>
<td>Gap shown in graph in between start/stop codon neighbors</td>
<td>Any integer value</td>
<td>3</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-[no]accid</td>
<td>boolean</td>
<td>Include to use sequence accession ID as query</td>
<td>Boolean value Yes/No</td>
<td>Yes</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-plot</td>
<td>toggle</td>
<td>Include to plot result</td>
<td>Toggle value Yes/No</td>
<td>No</td>
</tr>

</table>


<h2>Input file format</h2>

<p>
   The database definitions for following commands are available at<br />
   http://soap.g-language.org/kbws/embossrc<br />
<br />
   gviewcds reads one or more nucleotide sequences.<br />
<br />

</p>

<h2>Output file format</h2>

<p>
   The output from gviewcds is to a plain text file or the EMBOSS graphics device.<br />
<br />
   File: nc_000913.gviewcds<br />
<br />
Sequence: NC_000913<br />
position,A,T,G,C<br />
1,28.20,27.60,22.18,22.02<br />
2,26.05,26.81,23.06,24.08<br />
3,27.34,27.37,23.94,21.35<br />
4,26.28,28.83,23.01,21.88<br />
5,26.72,28.22,22.18,22.88<br />
6,26.42,26.72,24.96,21.90<br />
7,27.21,28.66,21.95,22.18<br />
8,25.47,28.39,23.06,23.08<br />
<br />
   [Part of this file has been deleted for brevity]<br />
<br />
400,26.60,27.44,22.67,23.27<br />
401,24.38,26.63,25.05,23.92<br />
402,25.03,26.37,23.71,24.87<br />
403,25.96,27.53,22.53,23.96<br />
404,26.63,25.52,24.17,23.66<br />
405,25.68,26.26,23.50,24.54<br />
406,24.94,26.86,23.92,24.26<br />
407,25.54,26.28,23.73,24.43<br />
408,25.28,26.93,24.38,23.39<br />
409,26.63,26.46,22.32,24.57<br />
<br />
<br />

</p>

<h2>Data files</h2>

<p>
None.
</p>

<h2>Notes</h2>

<p>
None.
</p>

<h2>References</h2>

<pre>
   Arakawa, K., Mori, K., Ikeda, K., Matsuzaki, T., Konayashi, Y., and
      Tomita, M. (2003) G-language Genome Analysis Environment: A Workbench
      for Nucleotide Sequence Data Mining, Bioinformatics, 19, 305-306.

   Arakawa, K. and Tomita, M. (2006) G-language System as a Platform for
      large-scale analysis of high-throughput omics data, J. Pest Sci.,
      31, 7.

   Arakawa, K., Kido, N., Oshita, K., Tomita, M. (2010) G-language Genome
      Analysis Environment with REST and SOAP Web Service Interfaces,
      Nucleic Acids Res., 38, W700-W705.

</pre>

<h2>Warnings</h2>

<p>
None.
</p>

<h2>Diagnostic Error Messages</h2>

<p>
None.
</p>

<h2>Exit status</h2>

<p>
It always exits with a status of 0.
</p>

<h2>Known bugs</h2>

<p>
None.
</p>

<h2>See also</h2>

<table border cellpadding=4 bgcolor="#FFFFF0"><tr><th>Program name</th>
<th>Description</th></tr>

<tr>
<td><a href="gbase_counter.html">gbase_counter</a></td>
<td>Creates a position weight matrix of oligomers around start                  codon</td>
</tr><tr>
<td><a href="gbase_z_value.html">gbase_z_value</a></td>
<td>Extracts conserved oligomers per position using Z-score</td>
</tr>

</table>

<h2>Author(s)</h2>

<pre>
   Hidetoshi Itaya (celery@g-language.org)
   Institute for Advanced Biosciences, Keio University
   252-0882 Japan

  Kazuharu Arakawa (gaou@sfc.keio.ac.jp)
  Institute for Advanced Biosciences, Keio University
  252-0882 Japan</pre>

<h2>History</h2>

   2012 - Written by Hidetoshi Itaya

<h2>Target users</h2>

   This program is intended to be used by everyone and everything, from
   naive users to embedded scrips.

<h2>Comments</h2>

   None.

