<?xml version="1.0" encoding="iso-8859-1"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <title>GEMBASSY</title> 
    <meta name="keywords" lang="en-uk" content="EMBOSS, European
    Molecular Biology Open Source Software Suite, bioinformatics, 
    EBI, European Bioinformatics Institute, Molecular Biology, 
    Computational Biology, Open Source, Software, Database" />
  </head>

<body bgcolor="#ffffff" style="background-color=#ffffff; body-background-color=#ffffff">
<link rel="stylesheet" href="../../stylesheets/EMBOSS.css" type="text/css" >

<!-- BEGIN PAGE HEADING -->
<div id="container">
<div id="header">  <a href="../index.html"> <img border="0" src="../../images/emboss.jpg" > </a> </div>
<!--#include virtual="../../header.inc" -->
<!-- END PAGE HEADING -->
    
<!--BEGIN MAIN CONTENT -->
<h1>GEMBASSY</h1>

<hr> 

The GEMBASSY package is intended to achieve higher usability of the
G-language SOAP service.


<p>

<h3><A NAME="gembassy">Applications</A> in the <a
href="">current gembassy release</a></h3>

<table border cellpadding=4 bgcolor="#FFFFF0">

<tr>
<th>Program name</th>
<th>Description</th>
</tr>

<tr>
<td><a href="gaminoinfo.html">gaminoinfo</a></td>
<td>
Prints out basic amino acid sequence statistics
</td>
</tr>
<tr>
<td><a href="gbaseinfocontent.html">gbaseinfocontent</a></td>
<td>
Calculates and graphs the sequence conservation using information content
</td>
</tr>
<tr>
<td><a href="gbui.html">gbui</a></td>
<td>
Calculates base usage indices for protein-coding sequences
</td>
</tr>
<tr>
<td><a href="gcai.html">gcai</a></td>
<td>
Calculate codon adaptation index (CAI) for each gene
</td>
</tr>
<tr>
<td><a href="gcbi.html">gcbi</a></td>
<td>
Calculates the codon bias index (CBI)
</td>
</tr>
<tr>
<td><a href="gcgr.html">gcgr</a></td>
<td>
Create a Chaos Game Representation of a given sequence
</td>
</tr>
<tr>
<td><a href="gfindoriter.html">gfindoriter</a></td>
<td>
Predict the replication origin and terminus in bacterial genomes
</td>
</tr>
<tr>
<td><a href="ggcsi.html">ggcsi</a></td>
<td>
GC Skew Index: an index for strand-specific mutational bias
</td>
</tr>
<tr>
<td><a href="ggcskew.html">ggcskew</a></td>
<td>
Calculate the GC skew of the given genome
</td>
</tr>
<tr>
<td><a href="ggcwin.html">ggcwin</a></td>
<td>
Calculate the GC content along the given genome
</td>
</tr>
<tr>
<td><a href="ggeneskew.html">ggeneskew</a></td>
<td>
Calculate the gene strand bias of the given genome
</td>
</tr>
<tr>
<td><a href="ggenomicskew.html">ggenomicskew</a></td>
<td>
Calculate the GC skew in different regions of the given genome
</td>
</tr>
<tr>
<td><a href="gicdi.html">gicdi</a></td>
<td>
Calculates the intrinsic codon deviation index (ICDI)
</td>
</tr>
<tr>
<td><a href="gphx.html">gphx</a></td>
<td>
Identify predicted highly expressed (PHX) genes
</td>
</tr>
<tr>
<td><a href="gqueryarm.html">gqueryarm</a></td>
<td>
Get the replication arm name (left or right) from the given position
</td>
</tr>
<tr>
<td><a href="gquerystrand.html">gquerystrand</a></td>
<td>
Get the strand name (leading or lagging) from the given position
</td>
</tr>
<tr>
<td><a href="greporiter.html">greporiter</a></td>
<td>
Get the positions of replication origin and terminus
</td>
</tr>
<tr>
<td><a href="gseq2png.html">gseq2png</a></td>
<td>
Converts a sequence to PNG image
</td>
</tr>
<tr>
<td><a href="gseqinfo.html">gseqinfo</a></td>
<td>
Prints out basic nucleotide sequence statistics
</td>
</tr>
<tr>
<td><a href="gshuffleseq.html">gshuffleseq</a></td>
<td>
Create randomized sequence with conserved k-mer composition
</td>
</tr>
</table>

<hr>
</div>
</BODY>
</HTML>
